---
{"dg-publish":true,"dg-permalink":"molecular bioscience","permalink":"/molecular bioscience/","tags":["Tagless"],"noteIcon":""}
---

<style id="Force_Custom_Fonts" type="text/css">@font-face{font-style:normal;font-family:"Merriweather";src:local("Merriweather")}@font-face{font-style:bolder;font-family:"Merriweather";src:local("Merriweather")}@font-face{font-style:normal;font-family:"Merriweather";src:local("Merriweather");unicode-range:U+0-FF,U+2E80-9FFF,U+F900-FAFF,U+FE30-FE4F,U+20000-2FA1F}@font-face{font-style:bolder;font-family:"Merriweather";src:local("Merriweather");unicode-range:U+0-FF,U+2E80-9FFF,U+F900-FAFF,U+FE30-FE4F,U+20000-2FA1F}@font-face{font-style:normal;font-family:"Merriweather";src:local("Merriweather");unicode-range:U+0-FF}@font-face{font-style:bolder;font-family:"Merriweather";src:local("Merriweather");unicode-range:U+0-FF}:not(pre):not(code):not(textarea):not(tt):not(kbd):not(samp):not(var){font-family:"Merriweather"!important}pre,code,textarea,tt,kbd,samp,var{font-family:monospace!important}pre *,code *,textarea *,tt *,kbd *,samp *,var *{font-family:monospace!important}</style>


# <center><span style="color:#0055A2">Molecular Biosciences</span></center>



## DNA Replication
![Pasted image 20251023055318.png](/img/user/images/Pasted%20image%2020251023055318.png)
Two antiparallel strands are held together by hydrogen bonding between complimentary bases
### Process
![Pasted image 20251023043655.png](/img/user/images/Pasted%20image%2020251023043655.png)
The double helix is unwound and the complementary strands are separated by the enzyme [[#helicase]], creating what is known as the [[#replication fork]]. Following this fork, [[#primase]] and [[#polymerase]] begin to act in order to create a new [[#Complimentary DNA|complimentary strand]]. 
These enzymes can only work in the 5' to 3' direction so once unwound: 
	The leading strand undergoes a continuous replication process— since it has 3' to 5' directionality, the polymerase assembling the leading strand can follow the replication fork without interruption. 
	The lagging strand cannot be created in a continuous fashion— because its template strand has 5' to 3' directionality—so the polymerase works backwards from the [[#DNA Replication fork|replication fork]] causing pauses in the process as the primase and polymerase move in the opposite direction of the fork. The enzymes stop and start again while the DNA helicase breaks the strands apart. Once the [[#okazaki fragments]] are made, DNA ligase connects them into a single, continuous strand. 
	
The entire replication process is considered "semi-discontinuous".


### Types of DNA Structure
![](https://i.imgur.com/xrcNsTh.jpeg)

| Geometry attribute: | **A-form**   | **B-form**   | **Z-form**  |
| ------------------- | ------------ | ------------ | ----------- |
| Helix sense         | right-handed | right-handed | left-handed |
| Repeating unit      | 1 bp         | 1 bp         | 2 bp        |
| Rotation/bp         | 32.7°        | 34.3°        | 60°/2       |
| Mean bp/turn        | 11           | 10.5         | 12          |

#### A-DNA
https://upload.wikimedia.org/wikipedia/commons/transcoded/1/14/Adna3.ogv/Adna3.ogv.480p.vp9.webm
shorter, more compact helical structure 
base pairs are not perpendicular to the helix-axis
DNA is driven into the A form when under dehydrating conditions.
#### B-DNA:  
The most common form of DNA and the one usually depicted.  
base pairs perpendicular to the helix-axis
#### Z-DNA
https://upload.wikimedia.org/wikipedia/commons/f/f5/Z-DNA_orbit_animated_small.gif
Major and minor grooves arent well-defined 
base pairs Zig-zag relative to the helix-axis
occasionally induced by biological activity and then quickly disappears.
#### Supercoiled DNA:  
DNA supercoiling refers to the amount of twist in a particular DNA strand— which determines the amount of strain on it. A given strand may be "positively supercoiled" or "negatively supercoiled" (more or less tightly wound). The amount of a strand's supercoiling affects a number of biological processes, such as compacting DNA and regulating access to the genetic code . Certain enzymes, such as [[#topoisomerase]], change the amount of DNA supercoiling to facilitate functions. The amount of supercoiling in a given strand is in reference to "relaxed B-form" DNA. 
DNA becomes temporarily positively supercoiled when it is being replicated or transcribed
- This is a more compact structure that forms when the DNA helix is subjected to strain by being overwound or underwound.  
- Plays a role in the compaction of DNA in cells and also has implications in DNA replication and transcription.

Circlular DNA Coiling ![Pasted image 20251023052303.png](/img/user/images/Pasted%20image%2020251023052303.png)
Linear DNA Coiling ![Pasted image 20251023052317.png](/img/user/images/Pasted%20image%2020251023052317.png)
<sub>The twist is the number of helical turns in the DNA and the writhe is the number of times the double helix crosses over on itself (these are the supercoils) </Sub>

### 1957: Meselson–Stahl experiment
Confirmation that DNA is semi-conservative
![ExperimentsThatChanged3.jpg](/img/user/images/ExperimentsThatChanged3.jpg)

### Enzymes Required for DNA Replication  
1. [[#helicase]]
2. [[#Topoisomerase]]
3. [[#Primase]]
4. [[#Polymerase]] 
5. [[#Ligase]]

##### helicase 
![DNA_Helicase_DNA_Replication.png](/img/user/images/DNA_Helicase_DNA_Replication.png)
Helicases are motor proteins that move incrementally along a nucleic acid strand, separating the two hybridized nucleic acid strands (via the energy gained from ATP hydrolysis.) 
##### Topoisomerase ![Catalytic_mechanisms_of_Topoisomerases.png](/img/user/images/Catalytic_mechanisms_of_Topoisomerases.png)
DNA topoisomerases are enzymes that catalyze changes in the topological state of DNA, interconverting relaxed and supercoiled forms, linked (catenated) and unlinked species, and knotted and unknotted DNA.
bind to DNA and cut the sugar-phosphate backbone of either one (type I topoisomerases) or both (type II topoisomerases) of the DNA strands allowing the DNA to be untangled or unwound before the DNA backbone is resealed
<sub>Tension in a DNA helix will naturally be relived when the backbone is cut</sub>

#### Primase 
![Pasted image 20251023060418.png](/img/user/images/Pasted%20image%2020251023060418.png)
synthesizes RNA primer to initiate DNA replication ( carried out by DNA Polymerase) 
- DNA polymerase adds on to 3’
- Adds RNA primers to the DNA (which are later removed by [[#exonuclease]] tand replaced with DNA)
#### Polymerase 
Synthesis of new strand of DNA complementary to the template DNA
DNA polymerase has exonuclease activity and cannot start polymerizing directly
Primase has no self-correcting mechanism but can initiate polymerization... introduces many errors that are then removed by DNA Pol.
- A bacterium or eukaryotic cell has several different DNA polymerase enzymes  
- One DNA polymerase (a DNA replicase) undertakes semiconservative replication; the others are involved in repair reactions
Only one DNA polymerase is the replication enzyme.
![](https://i.imgur.com/AxFKKx0.png)
##### DNA polymerase 5’->3’ direction  
- E. coli has Three types of polymerase  
- I mainly used for filling in small DNA segments during repair or replication (removed RNA primer (5’-3’ exonuclease)  
- II precise function unclear but may substitute for DNA polymerase I  
- III the main enzyme of DNA replication (5’-3’ Polymerase)

####  Ligase
facilitates joining of DNA strands by formation of phosphodiester bonds 
Following the removal or the RNA primer by DNA Pol I (5’-3’ exonuclease activity) DNA Ligase catalyses the formation of a phosphodiester bond between the 3’-hydroxyl group at the end of one DNA chain and the 5’-phosphoryl group at the end of another.

#### Exonuclease


### Origin of replication
![](https://i.imgur.com/xkaYAWC.png)

DNA replication starts at specific locations: Origin of Replication in Eukaryotes
2 Key problems to overcome:  
- Replication must occur with extreme accuracy  
- Every nucleotide must be replicated only once  
Origins od replication in procaryotes have well defined consensus sequences, In eukaryotes they are less conserved
### Process of DNA Replication
1. Initiation
2. Elongation
3. Termination
#### Initiation
Replication initiates when a protein complex binds to the origin and “melts” the DNA there (opens up a replication bubble)
![](https://i.imgur.com/YmUrSlx.png)



##### Single-strand DNA-binding proteins AKA
Helix destabilizing proteins bind to the single stranded DNA behind helicase to prevent the DNA strands reforming double strands This also prevents hairpins forming and Still allows DNA polymerase to recognize the individual basepairs:
![](https://i.imgur.com/mkrgGUL.png)



#### Separate Eukaryotic DNA Polymerases undertake Initiation and Elongation
- A replication fork has one complex of DNA polymerase α/primase, one complex of DNA polymerase δ, and one complex of DNA polymerase ε.  
- The DNA polymerase α/primase complex initiates the synthesis of both DNA strands.  
- DNA polymerase ε elongates the leading strand and a second DNA polymerase δ elongates the lagging strand.
For DNA synthesis to proceed, the DNA double helix must be opened up (“melted”) ahead of the replication fork so that the incoming deoxyribonucleoside triphosphates can form base pairs with the template strands. However, the DNA double helix is very stable under physiological conditions; the base pairs are locked in place so strongly that it requires temperatures approaching that of boiling water to separate the two strands in a test tube. For this reason, two additional types of replication proteins—DNA helicases and single-strand DNA-binding proteins—are needed to open the double helix and provide the appropriate single-strand DNA templates for the DNA polymerase to copy. DNA helicases use ATP to propel themselves rapidly along a DNA single strand. When they encounter a region of double helix, they continue to move along their strand, thereby prying apart the helix at rates of up to 1000 nucleotide pairs per second Helicase needs some single stranded DNA to get started This is provided by cdc6 and cdt1
Idk Where this is supposed to go 
![](https://i.imgur.com/ZTCLhGv.png)

#### Replisome: Bacterial Vs Eukaryote Names of Enzymes
![](https://i.imgur.com/pEN8t8s.png)

| Ganeral                    | bacterial | eukaryotic                    |
| -------------------------- | --------- | ----------------------------- |
| replicative DNA Polymerase | DNA G     | DNA Polymerase Alpha/ PRimase |
| Sliding Clamp              | Beta/DNAN | PCNA                          |
| Sliding Clamp Loader       | T-Complex | RF-C                          |
| Helicase                   | DNAB      | CMG Complex                              |


### DNA Polymerase Makes mistakes at the replication fork  
- 1 per 100 thousand base pairs  
- But it can correct them!  
- Pol III has 3’ to 5’ exonuclease activity (correcting as it goes)  
- Error rate now every billion base pairs (1 in every 109)  
- Pol II has 5’ to 3’ exonuclease activity (eg for removing RNA primer)
If the DNA polymerase did nothing when a mispairing occurred between an incoming deoxyribonucleoside triphosphate and the DNA template, the wrong nucleotide would often be incorporated into the new DNA chain, producing frequent mutations. The high fidelity of DNA replication depends not only on the initial base-pairing but also on several “proofreading” mechanisms. The error-correcting reaction, known as exonucleolytic proofreading, occurs immediately after an incorrect nucleotide is covalently added to the growing chain. DNA polymerase enzymes require a previously formed, base-paired 3ʹ-OH end of a primer strand to allow elongation. Those DNA molecules with a mismatched (improperly base-paired) nucleotide at the 3ʹ-OH end of the primer strand are not effective as templates because the polymerase has difficulty extending such a strand. DNA Pol correct such a mismatched primer strand by means of a separate catalytic site. This 3ʹ-to-5ʹ proofreading exonuclease clips off any unpaired or mispaired residues at the primer terminus, continuing until enough nucleotides have been removed to regenerate a correctly base-paired 3ʹ-OH terminus that can prime DNA synthesis.
![](https://i.imgur.com/P0WuKiX.png)
Strand-directed mismatch repair Removes errors not fixed by DNA pol exonuclease:  
Identifies distortions in DNA caused by the presence of no-complimentary bases
![](https://i.imgur.com/nfdszzq.png)
Figure 5.19 Strand-directed mismatch repair. (A) The two proteins shown are present in both bacteria and eukaryotic cells:  
MutS binds specifically to a mismatched base pair, while MutL scans the nearby DNA for a nick. Once MutL finds a nick, it triggers the degradation of the nicked strand all the way back through the mismatch. Because nicks are largely confined to newly replicated strands in eukaryotes, replication errors are selectively removed. In bacteria, an additional protein in the complex (MutH) nicks unmethylated (and therefore newly replicated) GATC sequences, thereby beginning the process illustrated here. In eukaryotes, MutL contains a DNA nicking activity that aids in the removal of the damaged strand. (B) The structure of the MutS protein bound to a DNA mismatch. This protein is a dimer, which grips the DNA double helix as shown, kinking the DNA at the mismatched base pair. It seems that the MutS protein scans the DNA for mismatches by testing for sites that can be readily kinked, which are those with an abnormal base pair. (PDB code: 1EWQ.) Alberts Chapter 5
### There are 100s of gene products (proteins) involved in DNA  
repair  
Inheriting mutations in genes required DNA repair result in serious syndromes
6 BILLION BASE PAIRS
![](https://i.imgur.com/Ve0pVoU.png)
### Controlling the Direction of Mismatch Repair
- The mut genes code for a mismatch-repair system that deals with mismatched base pairs.  
- There is a bias in the selection of which strand to replace at mismatches  
 - The strand lacking methylation at a hemimethylated GATC/CTAG is usually replaced– because this is the “new” strand  
- This repair system is used to remove errors in a newly synthesized strand of DNA. At G-T and C-T mismatches, the T is preferentially removed.  
mutator : a gene whose mutation results in an increase in the basal level of  
mutation of the genome. Such genes are often code for proteins that are  
involved in repairing damaged DNA.
![](https://i.imgur.com/mqNMdIN.png)
![](https://i.imgur.com/fiOikzf.png)
![](https://i.imgur.com/nTGSZVd.png)

### Spontaneous DNA base changes occur after polymeriza
The DNA of each human cell loses about 18,000 purine bases (adenine and guanine) every day because their N-glycosyl linkages to deoxyribose hydrolyze, a spontaneous reaction called depurination.  
Spontaneous deamination of cytosine to uracil in DNA occurs at a rate of about 100 bases per cell per day
![](https://i.imgur.com/tzaj0Zx.png)
![](https://i.imgur.com/ObLvUay.png)

### What causes mutations  
Chemical mutagens: Deamination (induced by HNO2, nitrous acid)
![](https://i.imgur.com/ICzjWeu.png)
![](https://i.imgur.com/PO2tjqp.png)
![](https://i.imgur.com/uR5QXGD.png)
![](https://i.imgur.com/Gap8c59.png)

#### Base dimerization
![](https://i.imgur.com/ytiItnx.png)
![](https://i.imgur.com/CpT9k9P.png)

#### Base excision Repair (BER)  
Involves a battery of enzymes called DNA glycosylases. Each glycosylase recognize a specific type of altered base in DNA and catalyze its hydrolytic removal. There are at least six types of these enzymes, including those that remove deaminated Cs, deaminated As, different types of alkylated or oxidized bases, bases with opened rings, and bases in which a carbon–carbon double bond has been accidentally converted to a carbon–carbon single bond. an altered base detected by an enzyme-mediated “flipping-out” of the altered nucleotide from the helix, which allows the DNA glycosylase to probe all faces of the base for damage These enzymes travel along DNA using base-flipping to evaluate the status of each base. Once an enzyme finds the damaged base that it recognizes, it removes that base from its sugar DNA glycosylase enzymes recognizes specific inappropriate bases.  
Each enzyme cleaves the glycosyl bond that connects a particular recognized base (yellow) to the backbone sugar, removing it from the DNA.  
The “missing tooth” created by DNA glycosylase action is recognized by an enzyme called AP endonuclease (AP for apurinic or apyrimidinic, endo to signify that the nuclease cleaves within the polynucleotide chain), which cuts the phosphodiester backbone, after which the resulting gap is repaired by DNA Polymerase: Pol I in bacterial Pol β (beta), δ (delta) and ε (epsilon) in Eucaryotes
#### Nucleotide excision Repair (NER)  
Repairs the damage caused by almost any large change in the structure of the DNA double helix. Such “bulky lesions” include:  
Those created by the covalent reaction of DNA bases with large hydrocarbons (such as the carcinogen benzopyrene, found in tobacco smoke, coal tar, and diesel exhaust) Various pyrimidine dimers (T-T, T-C, and C-C) caused by sunlight. A large multienzyme complex scans the DNA for a distortion in the double helix, rather than for a specific base change. Once it finds a lesion, it cleaves the phosphodiester backbone of the abnormal strand on both sides of the distortion DNA helicase peels away the single-strand oligonucleotide containing the lesion. The large gap produced in the DNA helix is then repaired by DNA polymerase and DNA ligase
#### Excision Repair Systems in E. coli  
The term "Uvr" is derived from "UV radiation sensitive," as mutations in these genes render the bacteria more sensitive to DNA damage caused by ultraviolet
- The Uvr system makes incisions ~12 bases apart on both sides of damaged DNA, removes the DNA between them, and resynthesizes new DNA.
- incision : an endonuclease recognizes the damaged area in the DNA, and isolates it by cutting the DNA strand on both sides of the damage.
- excision : a 5’-3’ exonuclease removes a stretch of the damaged strand.
- Excision repair systems vary in their specificity, but share the same general features. Each system removes mispaired or damaged bases from DNA and then synthesizes a new stretch of DNA to replace them.
![](https://i.imgur.com/61y7njO.png)
In humans, the equivalent system to the Uvr-mediated nucleotide excision repair (NER) pathway involves different and more numerous components than in prokaryotes.  
The main components involved in human NER:  
1. Damage Recognition:  
- XPC (in complex with RAD23B and CETN2) recognizes the DNA damage globally throughout the genome.  
1. Damage Verification: Once the damage is recognized, XPA verifies the presence of the damage and stabilizes the DNA around the lesion. RPA (replication protein A) also binds to the single-stranded DNA regions to stabilize them.  
2. DNA Unwinding: The TFIIH complex, which contains the helicases XPB and XPD, unwinds the DNA around the lesion to allow for incision.  
3. Incision: The endonucleases XPF-ERCC1 and XPG introduce cuts in the DNA strand containing the damage. XPF-ERCC1 makes the incision 5' to the lesion, and XPG makes the incision 3' to the lesion.  
4. Gap-filling and Ligation: After the removal of the damaged oligonucleotide, DNA synthesis fills in the gap, primarily by DNA polymerase δ or ε in conjunction with the replication factors RFC and PCNA. DNA ligase I or III then seals the nick, completing the repair.  
While the core process is similar to the prokaryotic Uvr system, the human NER pathway involves more proteins and is adapted to the complexities of the eukaryotic genome and chromatin structure.
![](https://i.imgur.com/hZo7Gi0.png)
![](https://i.imgur.com/d2NveZH.png)
Double-strand breaks (DSBs) are among the most deleterious types of DNA damage, as they can lead to large deletions, chromosomal rearrangements, and genome instability if not repaired correctly. Various factors and conditions can induce DSBs:  
5. **Ionizing Radiation**: Sources like X-rays, gamma rays, and cosmic rays can cause DSBs directly by breaking both DNA strands or indirectly by generating reactive oxygen species (ROS) that attack the DNA.  
6. **Chemical Agents**:  
- **Anticancer drugs**: Some chemotherapy agents, such as bleomycin, work by inducing DSBs in tumor cells.  
- **Topoisomerase II poisons**: Drugs like etoposide and doxorubicin stabilize the intermediate complex formed by topoisomerase II during its action on DNA, resulting in DSBs.  
1. **Ultraviolet (UV) Radiation**: While UV primarily induces pyrimidine dimers, if two closely spaced lesions are present on opposite DNA strands, they can be converted to DSBs during repair processes.  
2. **Reactive Oxygen Species (ROS)**: By-products of cellular metabolism, especially during oxidative phosphorylation, ROS can attack DNA and generate various types of damage, including DSBs.  
3. **Replication Fork Collapse**: Obstacles to DNA replication, such as unrepaired DNA lesions or tightly bound protein complexes, can lead to stalling and subsequent collapse of replication forks, resulting in DSBs.  
4. **Mechanical Stress on DNA**: Overwinding or underwinding of DNA due to supercoiling can cause strain that leads to DSBs.  
5. **Retroviral Integration**: Some retroviruses integrate their DNA into the host genome, a process that can introduce DSBs.  
6. **Programmed DSBs in Meiosis**: DSBs are deliberately introduced during the early stages of meiosis to promote genetic recombination between homologous chromosomes.  
7. **Crispr-Cas Systems**: In genome-editing applications using the CRISPR-Cas system, the Cas nuclease introduces DSBs at specific genomic locations.  
If DSBs are not promptly and correctly repaired, they can lead to genomic rearrangements, mutations, and cell death. Fortunately, cells have evolved sophisticated repair mechanisms, such as homologous recombination and non-homologous end joining, to manage and rectify DSBs.

#### Homologous Recombination (HR)  
This is the preferred method for repairing DNA double-strand breaks that arise shortly after the DNA has been replicated, while the daughter DNA molecules are still held close together.  
In general, homologous recombination can be regarded as a flexible series of reactions, with the exact pathway differing from one case to the next. For example, the length of the repair “patch” can vary considerably depending on the extent of 5ʹ processing and new DNA synthesis, indicated in green.one of the damaged strands can use the complementary strand of the intact DNA duplex as a template for repair.  
The ends of the broken DNA are chewed back, or “resected,” by specialized nucleases to produce overhanging, single-strand 3ʹ ends. The next step is strand exchange, during which one of the single-strand 3ʹ ends from the damaged DNA molecule worms its way into the template duplex and searches it for homologous sequences through base-pairing. Once stable base-pairing is established, a DNA polymerase extends the invading strand by using the information provided by the undamaged template molecule, thus restoring the damaged DNA. The last steps—strand displacement, further repair synthesis, and ligation—restore the two original DNA double helices and complete the repair process.  
Strand exchange is facilitated by a protein called RecA in E. coli and Rad51 in eukaryotes
![](https://i.imgur.com/liD82zh.png)
![](https://i.imgur.com/Fj72Gl8.png)
#### Ataxia telangiectasia
- Recessive inherited syndrome (2 mutant copies of ATM gene)  
- Ataxia: Poor coordination  
- Telangiectasia: Small dilated blood vessels  
- Weakened immune system  
- Radiosensitive- High risk of lymphoma and leukemia

<span style="color:rgb(75, 236, 19)">##·T~~ranscription¶  
The·central·dogma··¶  
The·control·of·transcription·and·RNA··¶  
polymerase··¶  
#####·These·Are·All·Pictures·For·No·Reason¶  
![](https://i.imgur.com/8Uddv6Q.png)¶  
![](https://i.imgur.com/czFTeon.png)¶  
![](https://i.imgur.com/HE4r8Fz.png)¶  
¶  
###·Prokaryotic·RNA·polymerase¶  
#####·These·Are·Also·Pictures·For·No·Reason¶  
![](https://i.imgur.com/YaeBlaL.png)¶  
![](https://i.imgur.com/08tPnAS.png)¶  
![](https://i.imgur.com/o2ZsJSS.png)¶  
![](https://i.imgur.com/W3fNXWY.png)¶  
![](https://i.imgur.com/Uy3b3G0.png)¶  
![](https://i.imgur.com/yRdEeVJ.png)¶  
![](https://i.imgur.com/OulQFkF.png)¶  
![](https://i.imgur.com/I0ZI6Nm.png)¶  
![](https://i.imgur.com/Pckvkuf.png)¶  
![](https://i.imgur.com/mCMRNI2.png)¶  
![](https://i.imgur.com/ap9ssiF.png)¶  
![](https://i.imgur.com/EcmHMy6.png)¶  
![](https://i.imgur.com/k65ereM.png)¶  
![](https://i.imgur.com/KVXS0VS.png)¶  
![](https://i.imgur.com/XZOodTW.png)¶  
![](https://i.imgur.com/7xlW5qh.png)¶  
![](https://i.imgur.com/sV67sST.png)¶  
![](https://i.imgur.com/y5i0zfr.png)¶  
![](https://i.imgur.com/vRpZPlF.png)¶  
![](https://i.imgur.com/rs4QgHr.png)¶  
##·Translation¶  
Template·Directed·Protien·Synthesis¶  
![](https://i.imgur.com/1byN93e.png)¶  
![](https://i.imgur.com/NFrQod5.png)¶  
![](https://i.imgur.com/X365bk4.png)¶  
![](https://i.imgur.com/bfivhpz.png)¶  
![](https://i.imgur.com/7tEhpBK.png)¶  
![](https://i.imgur.com/d2WT13p.png)¶  
![](https://i.imgur.com/oJ6V5nI.png)¶  
![](https://i.imgur.com/IMGfjLB.png)¶  
![](https://i.imgur.com/rAcGgFH.png)¶  
![](https://i.imgur.com/89LsZK8.png)¶  
![](https://i.imgur.com/TOsHxzy.png)¶  
![](https://i.imgur.com/VwLarfi.png)¶  
![](https://i.imgur.com/6DQIxhM.png)¶  
![](https://i.imgur.com/oLxZblA.png)¶~~##·T~~ranscription¶  
The·central·dogma··¶  
The·control·of·transcription·and·RNA··¶  
polymerase··¶  
#####·These·Are·All·Pictures·For·No·Reason¶  
![](https://i.imgur.com/8Uddv6Q.png)¶  
![](https://i.imgur.com/czFTeon.png)¶  
![](https://i.imgur.com/HE4r8Fz.png)¶  
¶  
###·Prokaryotic·RNA·polymerase¶  
#####·These·Are·Also·Pictures·For·No·Reason¶  
![](https://i.imgur.com/YaeBlaL.png)¶  
![](https://i.imgur.com/08tPnAS.png)¶  
![](https://i.imgur.com/o2ZsJSS.png)¶  
![](https://i.imgur.com/W3fNXWY.png)¶  
![](https://i.imgur.com/Uy3b3G0.png)¶  
![](https://i.imgur.com/yRdEeVJ.png)¶  
![](https://i.imgur.com/OulQFkF.png)¶  
![](https://i.imgur.com/I0ZI6Nm.png)¶  
![](https://i.imgur.com/Pckvkuf.png)¶  
![](https://i.imgur.com/mCMRNI2.png)¶  
![](https://i.imgur.com/ap9ssiF.png)¶  
![](https://i.imgur.com/EcmHMy6.png)¶  
![](https://i.imgur.com/k65ereM.png)¶  
![](https://i.imgur.com/KVXS0VS.png)¶  
![](https://i.imgur.com/XZOodTW.png)¶  
![](https://i.imgur.com/7xlW5qh.png)¶  
![](https://i.imgur.com/sV67sST.png)¶  
![](https://i.imgur.com/y5i0zfr.png)¶  
![](https://i.imgur.com/vRpZPlF.png)¶  
![](https://i.imgur.com/rs4QgHr.png)¶  
##·Translation¶  
Template·Directed·Protien·Synthesis¶  
![](https://i.imgur.com/1byN93e.png)¶  
![](https://i.imgur.com/NFrQod5.png)¶  
![](https://i.imgur.com/X365bk4.png)¶  
![](https://i.imgur.com/bfivhpz.png)¶  
![](https://i.imgur.com/7tEhpBK.png)¶  
![](https://i.imgur.com/d2WT13p.png)¶  
![](https://i.imgur.com/oJ6V5nI.png)¶  
![](https://i.imgur.com/IMGfjLB.png)¶  
![](https://i.imgur.com/rAcGgFH.png)¶  
![](https://i.imgur.com/89LsZK8.png)¶  
![](https://i.imgur.com/TOsHxzy.png)¶  
![](https://i.imgur.com/VwLarfi.png)¶  
![](https://i.imgur.com/6DQIxhM.png)¶  
![](https://i.imgur.com/oLxZblA.png)¶~~</span>
## Tutorial analysis of transcription

### Introduction  
- Important insights into gene function can be gained by expression profiling, (which genes are switched on when).  
- In multicellular organisms such as plant species many genes are expressed in particular cell types or at certain developmental stages. Furthermore, mutating one gene can alter expression of others (master genes etc).  
- All this information helps to link genes into functional networks, and genes can be used as markers to define cellular states.
- Until quite recently, genes and their expression profiles have been studied one at a time. Therefore, defining networks has been almost impossible!  
- Since the mid 1990s technological advances have enabled us to study the expression profiles of thousands of genes simultaneously, culminating in global expression profiling where every gene in a particular genome is monitored in a single expt!!  
- This is performed at the RNA level (direct sequencing or DNA microarrays) or at the protein level ( by 2-D gel electrophoresis followed by aa sequencing).
- Global expression profiling produces a holistic view of cellular activity. Far different and more comprehensive than the 1970s to early 1990s technologies. However, some these older techniques are still of use today. Gene-by-gene techniques for expression analysis  
- Since the 1970s techniques have been available to monitor gene expression on an individual basis. Unfortunately, none of these techniques is particularly suited to global expression profiling these included:  
1) Northern blots/RNA dot blots  
2) RT-PCR  
3) Western blots  
4) Enzyme linked immunosorbent assay (ELISA)
5) In situ hybridisation (See figures 1- 10 )  
- Although there were some techniques developed during the late 1980s that were used for parallel analysis of multiple genes e.g. differential hybridisation. These procedures were technically demanding and very labour intensive.  
- For true global analysis, new technology with a high degree of automation, which allowed thousands or tens of thousands of genes to be assayed simultaneously with minimal effort needed to be developed!

### Global analysis of RNA expression  
- The transcriptome - the full complement of RNA molecules produced by a particular genome (Veculescu et al, 1997).  
- Because of gene regulation the transcriptome is never observed as a complete system in vivo. There is no situation in which all the genes in the genome are simultaneously expressed.  
- All cells transcribe a basic set of housekeeping genes whose activity is required at all times for elementary functions, but other luxury genes are expressed in a regulated manner , e.g. as part of the developmental programme or in response to an external stimulus.
- Similarly, post-transcriptional events such as splicing are also regulated events. Therefore, researchers use phrases such as ‘human brain transcriptome’ or ‘plant zygotic transcriptome’ to emphasise this.  
- A typical human cell is thought to express ~ 20,000 different mRNAs, some of which have housekeeping functions and some of which are more specialised. A proportion of these will be splice variants of the same primary transcript. Some will be abundant, some rare.  
- So, for a truly global perspective of RNA expression all transcripts must be quantified at the same time. This requires a highly parallel assay format which is both sensitive and selective!
- There are currently two main types of strategy used for global RNA expression analysis  
1) Hybridisation analysis with comprehensive non-redundant collections of DNA sequences immobilised on a solid support. These are known as DNA arrays.  
2) Direct sampling of sequences from source RNA populations or cDNA libraries, or from sequence databases derived therefrom.

### DNA array technology  
- DNA array technology is the method of choice for high-throughput RNA expression analysis (Bowtell, 1999; Granjeaud et al, 1999). Arrays comprise a series of DNA elements arranged as spots (features) in a grid pattern on a solid support. These arrayed targets are hybridised (interrogated) with a complex probe I.e. a probe comprising many different sequences, which is prepared from an RNA population from a particular cell type or tissue.
- The composition of the probe reflects the abundances of individual transcripts in the source RNA population. Under the conditions used, the intensity of the hybridisation signal for each feature represents the relative level of the corresponding transcript in the probe.  
- The use of such arrays allows simultaneous measurement of the relative levels of many transcripts, and the only intrinsic limitation of the technique is the number and nature of sequences represented on the array.  
- Since 1997, whole genome arrays have been available for many model species, thus providing a truly global view of gene expression!

#### Spotted DNA arrays  
- A spotted DNA array is made by spotting cDNA or DNA clones (or PCR amplified DNA clones) onto a solid support where they are immoblised.  
- The first high-density cDNA arrays, now termed macroarrays, were essentially gridded cDNA libraries (Primrose et al, 2001).  
- cDNAs stored in a matrix format in microtitre plates were transferred to nylon membranes in a precise grid pattern, allowing rapid identification of the clones corresponding to positive hybridisation signals.

- For expression analysis complete libraries could be hybridised with complex probes generating a ‘fingerprint’ specific to a particular RNA source. Technology development  
- Nylon macroarrays are generally 10 or 20 cm2 in size and the feature density is low, with typically 1-2mm btwn targets (10-100 targets per cm2).This has some advantages: the arrays are easy to manufacture (cheap) and they are also easy to use because standard hybridisation procedures are applicable.
- For these reasons macroarrays are still made by a number of commercial suppliers and the technology for in-house array production is readily available, involving simple robotic devices or even handheld arrayers. However, as with all technology, macroarrays have several major disadvantages:  
1) The low feature density limits the number of sequences that can be interrogated simultaneously
2) hybridisation must be carried out in a large volume using a radioactive probe; the results being obtained by autoradiography.

#### The rise of the microarray!  
- A breakthrough in spotted array technology came with the development of microarrays on glass slides (Schena et al, 1995, 1996). Because glass is non-porous and has very little autofluorescence, fluorescent probes can be used and they can be applied in very small hybridisation volumes. The greater resolution afforded by fluorescent probes allows feature density to be increased and a smaller hybridisation volume improves the rxn kinetics.  
- Therefore, more features can be analysed simultaneously with the same amt of probe without loss of sensitivity.
- Thus, glass arrays can be made with 5000 features cm2. In addition, several different fluorophores can be used to label different RNA populations which can be hybridised to the microarray simultaneously, allowing differential gene expression btwn samples to be monitored directly (Shalon et al, 1996)!  
- The most common strategy is to use Cy3 and Cy5 labelled probes (explain).

#### Practical considerations in the manufacture of glass slide microarrays  
- When glass microarrays were first devised in the mid 1990s they had to be bought from a manufacturer and were single use and very expensive.  
- The clones used for different organisms varies: genomic clones tend to be used for bacterial arrays because of the lack of introns as is the case for yeasts. In higher eukaryotes, where introns are larger cDNAs are used the use of partial cDNA sequences that exploit the differences btwn related clones and helps avoid cross-hybridisation.
- Most of the cDNA information that exists in databases is in the form of expressed sequence tags (ESTs) and these are a valuable resource for the manufacture of spotted arrays.  
- Fortunately, over the past few years the cost of producing and screening microarrays has begun to fall!  
- Microarray robots can now be built for £20, 000 and instructions for building arraying robots using simple and readily available components are available on the internet (see figure 16). Which will mean that this powerful technique is no longer beyond the reach of all but the most prestigious Universities.

#### Printed oligonucleotide chips  
- The alternative to a spotted DNA array is a high-density prefabricated chip. These are similar to DNA arrays in that they consist of gridded DNA targets which are interrogated by hybridisation. However, while DNA arrays consist of double-stranded clones or PCR products that may be up to several hundred base pairs in length, oligo chips contain single-stranded targets ranging from 25-70 nucleotides.  
- Oligo chips can be made in the same way as spotted DNA arrays, by robotically transferring chemically synthesised oligonucleotides from microtitre dishes to a solid support, where they are immobilised.
- However, the main advantage is that the max array density is increased up to ten fold if the oligos are printed directly onto the glass surface!

## Metabolism
Catabolism- Large molecule to small molecule for energy
Anabolism- Small molecule to large molecule using energy
![Pasted image 20251009135448.png](/img/user/images/Pasted%20image%2020251009135448.png)
![Pasted image 20251009140520.png](/img/user/images/Pasted%20image%2020251009140520.png)
![Pasted image 20251009135509.png](/img/user/images/Pasted%20image%2020251009135509.png)
![Pasted image 20251009135518.png](/img/user/images/Pasted%20image%2020251009135518.png)
![Pasted image 20251009135542.png](/img/user/images/Pasted%20image%2020251009135542.png)
### Idk What "This" Is

![Pasted image 20251009135610.png](/img/user/images/Pasted%20image%2020251009135610.png)

## Mitochondria
![Pasted image 20251009135757.png](/img/user/images/Pasted%20image%2020251009135757.png)
![Pasted image 20251009135919.png](/img/user/images/Pasted%20image%2020251009135919.png)
![Pasted image 20251009140537.png](/img/user/images/Pasted%20image%2020251009140537.png)
![Pasted image 20251009140721.png](/img/user/images/Pasted%20image%2020251009140721.png)
![Pasted image 20251009140730.png](/img/user/images/Pasted%20image%2020251009140730.png)
![Pasted image 20251009140804.png](/img/user/images/Pasted%20image%2020251009140804.png)
![Pasted image 20251009140814.png](/img/user/images/Pasted%20image%2020251009140814.png)![Pasted image 20251009140827.png](/img/user/images/Pasted%20image%2020251009140827.png)
![Pasted image 20251009140839.png](/img/user/images/Pasted%20image%2020251009140839.png)
![Pasted image 20251009140853.png](/img/user/images/Pasted%20image%2020251009140853.png)
![Pasted image 20251009140947.png](/img/user/images/Pasted%20image%2020251009140947.png)
![Pasted image 20251009141003.png](/img/user/images/Pasted%20image%2020251009141003.png)
![Pasted image 20251009141018.png](/img/user/images/Pasted%20image%2020251009141018.png)
![Pasted image 20251009141027.png](/img/user/images/Pasted%20image%2020251009141027.png)![Pasted image 20251009141042.png](/img/user/images/Pasted%20image%2020251009141042.png)
![Pasted image 20251009141058.png](/img/user/images/Pasted%20image%2020251009141058.png)
![Pasted image 20251009141111.png](/img/user/images/Pasted%20image%2020251009141111.png)
![Pasted image 20251009141123.png](/img/user/images/Pasted%20image%2020251009141123.png)
![Pasted image 20251009141135.png](/img/user/images/Pasted%20image%2020251009141135.png)
![Pasted image 20251009141148.png](/img/user/images/Pasted%20image%2020251009141148.png)
![Pasted image 20251009141202.png](/img/user/images/Pasted%20image%2020251009141202.png)
![Pasted image 20251009141213.png](/img/user/images/Pasted%20image%2020251009141213.png)










# Terminology
###### Replication fork
The replication fork is a structure that forms within the long helical DNA during DNA replication. It is produced by enzymes called helicases that break the hydrogen bonds that hold the DNA strands together in a helix. The resulting structure has two branching "prongs", each one made up of a single strand of DNA. These two strands serve as the template for the leading and lagging strands, which will be created as DNA polymerase matches complementary nucleotides to the templates; the templates may be properly referred to as the leading strand template and the lagging strand template.![Pasted image 20251023044620.png](/img/user/images/Pasted%20image%2020251023044620.png)

###### okazaki fragments
short sequences of DNA nucleotides which are synthesized discontinuously and later linked together by the enzyme DNA ligase to create the lagging strand during DNA replication
![Okazaki_fragment_EN.svg.png](/img/user/images/Okazaki_fragment_EN.svg.png)
###### Complimentary DNA
Complementary DNA (cDNA) is DNA that was reverse transcribed (via [[#reverse transcriptase]]) from an RNA (e.g., mRNA or miRNA). cDNA exists in both single-stranded and double-stranded forms and in both natural and engineered forms. 

###### microRNA
Micro ribonucleic acid (microRNA, miRNA, μRNA) are small, single-stranded, non-coding RNA molecules containing 21–23 nucleotides.[1] Found in plants, animals, and even some viruses, miRNAs are involved in RNA silencing and post-transcriptional regulation of gene expression.[2][3] miRNAs base-pair to complementary sequences in messenger RNA (mRNA) molecules,[4] then silence said mRNA molecules by one or more of the following processes:[1][5]
    Cleaving the mRNA strand into two pieces.
    Destabilizing the mRNA by shortening its poly(A) tail.
    Reducing translation of the mRNA into proteins.

In cells of humans and other animals, miRNAs primarily act by destabilizing the mRNA.[6][7] 






<center><sub>Done :)</sub></center>