---
{"dg-publish":true,"permalink":"/uni/2/molecular-biosciences/5-bc-005-molecular-biosciences-practical-techniques/molecular-biosciences-practical-techniques/","tags":["Tagless"],"noteIcon":""}
---

<style id="Force_Custom_Fonts" type="text/css">@font-face{font-style:normal;font-family:"Merriweather";src:local("Merriweather")}@font-face{font-style:bolder;font-family:"Merriweather";src:local("Merriweather")}@font-face{font-style:normal;font-family:"Merriweather";src:local("Merriweather");unicode-range:U+0-FF,U+2E80-9FFF,U+F900-FAFF,U+FE30-FE4F,U+20000-2FA1F}@font-face{font-style:bolder;font-family:"Merriweather";src:local("Merriweather");unicode-range:U+0-FF,U+2E80-9FFF,U+F900-FAFF,U+FE30-FE4F,U+20000-2FA1F}@font-face{font-style:normal;font-family:"Merriweather";src:local("Merriweather");unicode-range:U+0-FF}@font-face{font-style:bolder;font-family:"Merriweather";src:local("Merriweather");unicode-range:U+0-FF}:not(pre):not(code):not(textarea):not(tt):not(kbd):not(samp):not(var){font-family:"Merriweather"!important}pre,code,textarea,tt,kbd,samp,var{font-family:monospace!important}pre *,code *,textarea *,tt *,kbd *,samp *,var *{font-family:monospace!important}</style>


# <center><span style="color:#AAAAAA">Molecular Biosciences Practical Techniques</span></center>

![[Practical 1 PCR 2024.docx]]

## vector
capable of carrying a fragment of foreign DNA  
(Vector + foreign DNA = Recombinant DNA)
cDNA: Complementary DNA
	DNA Without Introns
OPR (Open Reading Frame) And CDS (Coding DNA Sequence) Are The Same Thing

Types of Vectors:
- Cloning Vectors 
	- isolate and amplify DNA
	![Pasted image 20251109222050.png](/img/user/images/Pasted%20image%2020251109222050.png)
	![pBluescript_II_SK(+).png](/img/user/images/pBluescript_II_SK(+).png)
- Expression Vectors
	- isolate and amplify DNA
	- express foreign DNA (gene) in host organism/cell (DNA RNA Protein)
![Pasted image 20251109220241.png](/img/user/images/Pasted%20image%2020251109220241.png)
![pcDNA3.1(-).png](/img/user/images/pcDNA3.1(-).png)
pcDNA+ MCS Is In A Different Order

Other Vectors:
- Bacteriophage
- Bacterial Artificail Chromosome (BAC)
![Pasted image 20251113001917.png](/img/user/images/Pasted%20image%2020251113001917.png)
- Yeast Artificial Chromosome (YAC)

## Gene Cloning
### Plasmid 
 dsDNA molecule capable of independent replication within a host organism (Bacteria).  

All plasmids contain:
- Origin of replication (ori)
- Charicteristic gene (Like amp<sup>r</sup>)
 
#### Plasmid Modifications
##### Multiple cloning site
A small sequence (~30-80bp) of DNA containing recognition sites for a number of common restriction enzymes (must be unique to the cloning site)

 pBluescript II Mcs:
Kpn1-XhoI-ClaI-HindIII-EcoRV-EcoRI-PstI-BamHI-XbaI--NotI-BstxI-SacI

##### screening system System
###### The LacZ selection system
The b-galactosidase gene, LacZ, metabolizes Xgal    Producing a blue product
Insertion of foreign DNA into the MCS destroys the LacZ gene    no blue product



## Restriction Enzymes
Found naturally in bacteria- protect against infection from bacteriophages
Cut DNA at specific, palindromic sites

Size of insert (bp) X amount of vector (ng)
Over
Size of vector (bp)
1:1

 1 unit of restriction enzyme is defined as the amount required to cut 1ug of DNA in 1hr.
Enzymes are not stable and degrade with time
They should be kept on ice at all times and stored @ -20OC.
By reducing the units of enzyme used or time incubated DNA can be partially digested.

EcoRI Cuts Once
SalI Cuts Twice
NcoI Cuts Thrice

## DNA ligase
catalyses the formation of a phosphodiester bond between the 3’-hydroxyl group at the end of one double stranded DNA chain and the 5’-phosphoryl group at the end of another.
To ensure the best chance of a successful ligation reaction we add more insert (SPINT2) than Vector (pCDNA3.1)
When preforming ligations we measure the amount of DNA used in ends:  
100ng of SPINT2 DNA contains 7X more ends than 100ng of pCDNA3.1 DNA

To calculate the ug of an insert required to equal the ends of a vector:

## Primer
Forward Primer Is The Same As The Target Primer
Reverse Primer Is Complimentary To The Forward DNA

Primers Made With 21 Base Pairs

Primer written: 5'  3'  

## SPINT2
SPINT2 Is Also Called HFG Activator Inhibitor Type 2 (HAI-2)
Hepatocyte growth factor (HFG)
MET   (AKA: HGF receptor)
**Kunitz-type protease inhibitor 2** is an [enzyme inhibitor](https://en.wikipedia.org/wiki/Enzyme_inhibitor "Enzyme inhibitor") that in humans is encoded by the _SPINT2_ [gene](https://en.wikipedia.org/wiki/Gene "Gene").[[5\|5]](https://en.wikipedia.org/wiki/SPINT2#cite_note-pmid9115294-5)[[6\|6]](https://en.wikipedia.org/wiki/SPINT2#cite_note-pmid9346890-6)[[7\|7]](https://en.wikipedia.org/wiki/SPINT2#cite_note-entrez-7) SPINT2 is a [transmembrane protein](https://en.wikipedia.org/wiki/Transmembrane_protein "Transmembrane protein") with two extracellular [Kunitz domains](https://en.wikipedia.org/wiki/Kunitz_domain "Kunitz domain") to inhibit [serine proteases](https://en.wikipedia.org/wiki/Serine_protease "Serine protease"). This gene is a presumed [tumor suppressor](https://en.wikipedia.org/wiki/Tumor_suppressor_gene "Tumor suppressor gene") by inhibiting HGF activator which prevents the formation of active [hepatocyte growth factor](https://en.wikipedia.org/wiki/Hepatocyte_growth_factor "Hepatocyte growth factor"). [Mutations](https://en.wikipedia.org/wiki/Mutation "Mutation") in SPINT2 could result in [congenital sodium diarrhea](https://en.wikipedia.org/w/index.php?title=Congenital_sodium_diarrhea&action=edit&redlink=1 "Congenital sodium diarrhea (page does not exist)") (CSD).

## Transformation
Genetic alteration of a cell following uptake and expression of exogenous genetic material (plasmids).
Naturally occurring: Horizontal gene transfer  
Modified by Biologists used as a powerful tool in research, Biotechnology and medicine

e.Coli are not naturally competent

Methods of transformation:
- Electroporation
- Heat Shock

Host Cell must be sensitive to the antibiotic
The vector carrying Antibiotic resistant gene confers resistant to the host cell

Preparation of calcium competent E.coli.  
Day 1  
Set up an overnight culture of E.coli  
Day 2  
1. Inoculate 20 ml of Luria broth with 0.75 ml of the O/N E.coli culture.  
2. Incubate, with aeration, for 90-100 minutes at 37°C (to an OD~O.6).  
3. Spin down the cells at 10 K for 30 seconds.  
4. Wash the cells in 5 ml of 5 mM NaCl and spin down at 10K for 5 minutes at 4°C.  
5. Resuspend the cells in 2 ml of 100 mM CaCl2 and incubate on ice for at least 20  
minutes.  
6. Spin the cells down at 10 K for 5 minutes at 4°C.  
7. Resuspend the cells in 1 ml of 100 mM CaC12


1.2a: Historic mini-prep protocol Solutions  
Solution GTE:               50 mM glucose, 25 mM Tris/HCl (pH 8.0), 10 mM EDTA  
Solution SDS/NaOH: 0.2 M NAOH, 1 % (w/v) SDS (sodium dodecyl sulphate)  
Solution KAc:                5 M Potassium Acetate (60 ml), Glacial acetic acid (11.5 ml),  
H2O (28.5 ml).  

## Expression of cloned genes in human cells
Cellular localisation [images]
Effect on Cell physiology
## Moles, Daltons and Constants
A Mole is the amount of a substance that contains the same number of elementary entities (atoms, molecules etc.) as 12g of Carbon.

Atomic Mass of Carbon= 12Da (The accepted unit of molecular Mass is the Dalton (Da or kDa)
1Mol of Carbon= 12g

The number of Carbon atoms in 1Mol (12g) is:  6.022 X1023 
This is the Avogadro constant: N<sub>A</sub> or L  
N<sub>A</sub> is the number if units of a substance per Mole.  
6.022 X1023 mol-1

((x/1.5)<sup>6</sup>)10<sup>-9</sup>ng
A/ng = Molecules/ng
### Calculations

Atomic Mass of Carbon= 12  
1Mol of Carbon= 12g  
6.022 X1023 atoms in 12g= 1mol  

Atomic Mass of Sodium= 22.99  
1Mol of Sodium= 22.99g  
6.022 X1023 atoms in 22.99g= 1mol

Atomic Mass of Sodium= 22.99  
Atomic Mass of Oxygen= 16  
Atomic Mass of hydrogen= 1.01  
Atomic Mass of sodium Hydroxide= 40.00  
1Mol of Sodium Hydroxide= 40.00g  
6.022 X1023 Molecules in 40.00g

The Molecular Mass of dsDNA  
1 Base pair approx. Average Mass:  
650Da  
1500bp (1.5kb) X650=975,000 Da  
=0.975X106 Da  
1X106 Da  
1mol= 1X106g =1000kg  
6.022X1023 Molecules in 1000kg  
6.022X1020 Molecules in 1kg (1000g)  
6.022X1017 Molecules in 1g  
6.022X1014 Molecules in 1mg (0.001g)  
6.022X1011 Molecules in 1ug (0.000001g)  
602,200,000,000  
(6.022X1023/1X1012=6.022X1011)

#### How many molecules of pBS are there in 100ng?
 1.5kb= 1X106 Da
(3/1.5=2)
 1 pBS molecule has a mass of (3/1.5)= 2X106 Da
1mole of pBS= 2X106g
2X106g= 6.02X1023molecules
1ug= 1X10-6g
So: 2X106g= 2X106 X 1X106= 2X1012ug
2X1012ug= 6.02X1023molecules
So: 6.02X1023/2X1012= 3X1011 molecules of pBS/ug
1000ng in 1ug: 3X1011/1000=3X108 molecules of pBS in 1ng

1ng= 1X10-9g  
So: 2X106g= 2X106 X 1X109= 2X1015ng

6.02X1023/2X1015=3X108 (1ng) X100= 3X1010 (100ng)

### Scoring successful Transformants (pcDNA3.1 control)

#### Number of transformants per ug of DNA
No. of colonies: 200
Total Volume: 1000ul  
100ul plated  
DNA conc=10ug/ml (/10=ng/ul)  
5ul used=50ng=0.05ug  
0.05ug plasmid added  
Used 100ul of a total 1000ul  
100/1000= 0.1  
0.1X0.05= 0.005ug DNA(=5ng)  
So: 0.005ug gives 200 colonies  
Then: 200 (colonies) X1.0 (ug)/0.005=  
40,000 successful transformants per ug  
plasmid  
(4.0X104 )

### Scoring successful Transformants
#### Number of transforming molecules per ug of plasmid
No. of colonies: 200
Plasmid pCDNA3.1  (5.427kb)
1.5kb= 1X106 Da  
1 pCDNA3.1 molecule has a mass of (5.427/1.5)=  
3.62X106 Da  
1mole of pCDNA3.1= 3.62X106 g  
3.62X106 g= 6.02X1023 molecules  
1ug= 1X10-6 g  
So: 3.62X106 g= 3.62X106 X 1X106 = 3.62X1012 ug  
3.62X1012 ug= 6.02X1023 molecules  
So: 6.02X1023 /3.62X1012 = 1.66X1011 molecules  
of pCDNA3.1/ug  
(0.005ug in 100ul on plate  
0.005 X 1.66X1011 = 8.3X108 molecules  
830,000,000)


#### Number of E.coli plated
No. of colonies: 200
Plasmid pCDNA3.1  
(5.427kb)
Compitent cells conc.=5X108 /ml  
Added 100ul (5X107 ), total vol=1000ul  
Used 100ul:5X107 X (100/1000) 0.1=  
5,000,000= 5X106 cells plated
Fraction of successfully transformed bacteria:  
5X106 /200= 1 in 25,000
8.3X108 plasmids plated (1.66X1011 /ug)  
Number of successful transformants:  
200  
8.3X108 /200= 4.15X106 molecules per  
successful transformation
1 in 4.15X106 Successfully transformed
(4.15X106 = 4.15 million)

### Scoring successful Transformants (Ligation)
#### Number of transformants per ug of DNA
No. of colonies: 200
Plasmid pCDNA3.1
(5.427kb)
+SPINT2 (0.8kb)
=6.23kb
Number of transformants per ug of DNA
Total Volume: 1000ul  
1000ul plated  
DNA conc=100ng of pCDNA3.1 in 20ul  
+15ng of SPINT2=115ng  
10ul used=58ng=0.058ug  
0.058ug plasmid added  
Used 1000ul of a total 1000ul  
So: 0.058ug gives 200 colonies  
Then: 200 (colonies) X1.0 (ug)/0.058=  
3,448 successful transformants per ug  
plasmid  
(3.450X103 )
#### Number of transforming molecules per ug of plasmid
No. of colonies: 200
Plasmid pCDNA3.1
(5.427kb)
+SPINT2 (0.8kb)
=6.23kb
1.5kb= 1X106 Da  
1 pCDNA3.1-SPINT2 molecule has a mass of  
(6.23/1.5)= 4.15X106 Da  
1mole of pCDNA3.1-SPINT2= 4.15X106 g  
4.15X106 g= 6.02X1023 molecules  
1ug= 1X10-6 g  
So: 4.15X106 g= 4.15X106 X 1X106 = 4.15X1012 ug  
4.15X1012 ug= 6.02X1023 molecules  
So: 6.02X1023 /4.15X1012 = 1.45X1011 molecules  
of pCDNA3.1-SPINT2/ug  
(0.058ug in 1000ul on plate  
0.058 X 1.45X1011 = 8.41X109 molecules)
#### Number of E.coli plated
No. of colonies: 200
Plasmid pCDNA3.1
(5.427kb)
+SPINT2 (0.8kb)
=6.23kb
Compitent cells conc.=5X108 /ml
Added 100ul (5X107 ), total vol=1000ul
Used 1000ul = 5X107 cells plated
Fraction of successfully transformed bacteria:
5X107 /200= 1 in 250,000
8.41X109 plasmids plated (1.45X1011 /ug)
Number of successful transformants:
200
8.41X109 /200= 4.2X107 molecules per
successful transformation
1 in 4.2X107 Successfully transformed
(4.2X107 = 42 million)

## Questions:  
Week 1  
Restriction digest of plasmids to prepare gene (insert) and expression  
vector for ligation.  
Formative Questions:  
Q1: What is the recognition sequence of the enzymes Xho1 and HindIII  
Q2: Would the restriction reaction work at a) 4OC or b) 95OC  
Week 2  
TRANSFORMATION OF E. COLI  
Q3: Why is it necessary to incubate the transformed bacterial cells in media that does  
not contain any antibiotics before plating them onto growth media containing the  
antibiotic ampicillin?  
Q4: Calculate the number of molecules of pcDNA3.1 in 50ng (Show your workings)  
-Analysis of agarose gel electrophoresis patterns (Dry-Lab book figure)  
Week 3  
Gel electrophoresis of restriction digests  
-Determine the size of your restriction fragments (calibration graph not  
required).  
P35/10  
10kb BAC  
+  
35kB insert=  
45,000bp  
A  
m  
p

Q5: what do the bands you see represent?  
Q6: What does the TBE do?  
Q7: How does ethidium bromide work?  
Q8: Express the transformation frequency for each batch as:  
- transformants per μg of DNA.  
- transforming molecules of DNA as a fraction of the number of molecules of input  
DNA.  
Q9:  
a) Which plate has the most/ least colonies?  
b) Why is this?/ is this what you expected?  
c) Does the number of colonies obtained on your ligation plate correlate well to the  
amount of DNA used in the transformation?  
Week 4  
Isolation of plasmid DNA from a culture of E. Coli cells (minipreps)  
Q10: What is the function of the GTE?  
Q11: What is the function of the SDS/NaOH?  
Q12: What is the function of the potassium acetate?  
Q13: What is the function of the isopropanol?  
Week 5  
Gel electrophoresis of cloned DNA restriction digests  
Q14: describe what you see in each of the three digestion lanes, what do they  
represent? Was your sub-cloning experiment successful? Include a fully labeled  
figure (a calibration curve is not required).


## Lost in translation
Plasmid small
Expression Vector
	p- Promoter
Differences Between Vectors
((x/1.5)<sup>6</sup>)10<sup>-9</sup>ng
A/ng = Molecules/ng
"complementary cOhesvive Pairs"
Incubate Constriction At 4°C  To Provide Time For Hydrogen Bonding 
	ATP Provides Energy Not Heat
	Call Wall And DNA Are Negatively Charged
	CaCl<sub>2</sub><sup>2+</sup> Makes The Bacteria "Chemically Competent"
Beta Lactinase Degrades Ampicillin

## Mark
### Tumor suppressor activity and epigenetic inactivation of hepatocyte growth factor activator inhibitor type 2/SPINT2 in papillary and clear cell renal cell carcinoma
#### Abstract

Following treatment with a demethylating agent, 5 of 11 renal cell carcinoma (RCC) cell lines showed increased expression of hepatocyte growth factor (HGF) activator inhibitor type 2 (HAI-2/SPINT2/Bikunin), a Kunitz-type protease inhibitor that regulates HGF activity. As activating mutations in the MET proto-oncogene (the HGF receptor) cause familial RCC, we investigated whether HAI-2/SPINT2 might act as a RCC tumor suppressor gene. We found that transcriptional silencing of HAI-2 in RCC cell lines was associated with promoter region methylation and HAI-2/SPINT2 protein expression was down-regulated in 30% of sporadic RCC. Furthermore, methylation-specific PCR analysis revealed promoter region methylation in 30% (19 of 64) of clear cell RCC and 40% (15 of 38) of papillary RCC, whereas mutation analysis (in 39 RCC cell lines and primary tumors) revealed a missense substitution (P111S) in one RCC cell line. Restoration of HAI-2/SPINT2 expression in a RCC cell line reduced in vitro colony formation, but the P111S mutant had no significant effect. Increased cell motility associated with HAI-2/SPINT2 inactivation was abrogated by treatment with extracellular signal-regulated kinase (ERK)/mitogen-activated protein kinase (MAPK) and phospholipase C-gamma inhibitors, but not by an inhibitor of atypical protein kinase C. These findings are consistent with frequent epigenetic inactivation of HAI-2/SPINT2, causing loss of RCC tumor suppressor activity and implicate abnormalities of the MET pathway in clear cell and papillary sporadic RCC. This information provides opportunities to develop novel targeted approaches to the treatment of RCC.

#### Introduction

Renal cell carcinoma (RCC) is a heterogeneous disorder with the majority (∼75%) of cases classified as clear cell (conventional) and the next most frequent subtype is papillary RCC (∼15% of all cases; ref. 1). Germline activating mutations in the MET proto-oncogene and inactivating mutations in the VHL tumor suppressor gene cause hereditary type 1 papillary RCC (HPRC1) and von Hippel-Lindau disease, respectively (2, 3). HPRC1-associated MET mutations activate MET signaling to promote cell growth and motility (4–7). The VHL tumor suppressor gene is inactivated by somatic mutation or promoter methylation in most sporadic clear cell RCC (8–11). However, somatic MET-activating mutations are uncommon (<10%) in sporadic papillary RCC and are not a feature of sporadic clear cell RCC (3). Thus, despite reports of increased expression of hepatocyte growth factor (HGF) and MET in RCC and synergy between the effect of VHL inactivation and increased MET signaling (12), direct genetic evidence for importance of the HGF/MET signaling in the pathogenesis of RCC is only present in a small minority of cases.

Methylation of CpG dinucleotides in the promoter regions of tumor suppressor genes producing transcriptional silencing plays a major role in many human cancers (13–16). The frequency of tumor suppressor gene inactivation by de novo methylation varies between tumor suppressor genes and between cancers. Thus, the 3p21.3 renal tumor suppressor gene, RASSF1A, is usually inactivated by epigenetic silencing and only rarely by somatic mutations (17–22), whereas the VHL tumor suppressor gene is inactivated more commonly by somatic mutation than by epigenetic silencing (8–11). For certain cancers, notably colorectal cancer, a subset of tumors may show extensive tumor suppressor gene promoter methylation (23). However, in a methylation profile analysis of RCC, we found promoter methylation of only a minority of tumor suppressor genes that were known to be inactivated in other tumor types (24, 25). This observation prompted us to investigate whether gene expression profiling of RCC cell lines treated with the demethylated agent 5-azacytidine might identify novel RCC tumor suppressor genes (26–28). During our investigations, we found silencing or down-regulation of HAI-2/SPINT2 expression in a number of RCC cell lines. HAI-2/SPINT2 (also known as bikunin) encodes Kunitz-type protease inhibitor that regulates HGF activity. Thus, HGF is secreted as an inactive proform and needs to be activated by HGF activator enzyme to initiate MET signaling. HAI-2/SPINT2 is an endogenous inhibitor of HGF activator enzyme and has been implicated in the pathogenesis of ovarian and hepatocellular carcinoma (29–31). In view of these findings, we proceeded to investigate whether epigenetic inactivation of SPINT2 might play a role in renal tumorigenesis.

#### Materials and Methods

**Patients and samples.** DNA from a total of 102 primary RCCs were analyzed and subdivided into 64 clear cell and 38 papillary RCCs. Protein lysates from 31 additional RCC tumors (29 clear cell RCC and 2 nonclear cell RCC) was also analyzed. Local ethics committees approved the collection of samples and informed consent was obtained from each patient.

**Cell lines, 5-azadeoxycytidine treatment, and microarray analysis.** RCC cell lines KTCL 26, RCC4, UMRC2, UMRC3, SKRC18, SKRC39, SKRC45, SKRC47, SKRC54, 786-0, and Caki-1 were routinely maintained in DMEM (Invitrogen, San Diego, CA) supplemented with 10% FCS at 37°C, 5% CO2. The demethylating agent 5-azadeoxycytidine (Sigma, Gillingham, Dorset, United Kingdom) was freshly prepared in double-distilled H2O and filter sterilized. Cell lines were plated in 75 cm2 flasks in DMEM supplemented with 10% FCS at differing densities, depending upon their replication factor, to ensure that both control and 5-azadeoxycytidine–treated lines reached ∼75% confluence at the point of RNA extraction. Twenty-four hours later, cells were treated with 5 μmol/L 5-azadeoxycytidine. The medium was changed 24 hours after treatment and then changed again after 72 hours. RNA was prepared 5 days after treatment using RNABee (AMS Biotechnology, Abingdonoxon, United Kingdom). RNA extracted from KTCL26, SKRC39, SKRC45, and SKRC47 ± 5-azadeoxycytidine was analyzed by microarray as previously described (32).

_HAI-2/SPINT2_ expression was detected by reverse transcription-PCR (RT-PCR) using the following primers: 5′-CAGCATCCACGACTTCTGCCTG-3′ and 5′-GGCGGTGCAGTATTCTTCATAG-3′.

Expression of _GAPDH_ was used as a control. The _GAPDH_ primers were 5′-TGAAGGTCGGAGTCAACGGATTTGGT-3′ and 5′-CATGTGGGCCATGAGGTCCACCAC-3′ The PCR cycling conditions for these reaction consisted of 5 minutes at 95°C followed by 30 cycles of 45 seconds of denaturation at 95°C, 45 seconds of annealing at 59°C, and 45 seconds of extension at 72°C. Semiquantitative analysis of _HAI-2/SPINT2_ expression was done using LabWorks software (Ultraviolet Products, Cambridge, United Kingdom).

**Bisulfite modification and methylation analysis.** Bisulfite DNA sequencing was done as described previously (24). Briefly, 0.5 to 1.0 μg of genomic DNA was denatured in 0.3 mol/L NaOH for 15 minutes at 37°C, and then unmethylated cytosine residues were sulfonated by incubation in 3.12 mol/L sodium bisulfite (pH 5.0; Sigma)/5 mmol/L hydroquinone (Sigma) in a thermocycler (Hybaid, Basingstore, Hampshire, United Kingdom) for 20 cycles of 30 seconds at 99°C and 15 minutes at 50°C. The sulfonated DNA was recovered using the Wizard DNA cleanup system (Promega, Southampton, United Kingdom) in accordance with the manufacturer's instructions. The conversion reaction was completed by desulfonating in 0.3 mol/L NaOH for 10 minutes at room temperature. The DNA was ethanol-precipitated and resuspended in water.

The _HAI-2/SPINT2_ CpG island was identified on the human genome browser and the _putative_ promoter region was predicted by Promoter Inspector software (Genomatix, Munich, Germany). The region was amplified from RCC cell lines and primary tumors using the following primers: SPINT2F (5′-TTTTYGGTATTAGGGGTGGGTTTAGGT-3′), SPINT2R (5′-CCAAAAAAAAACAACRATCCCAACAAAAC-3′), SPINT2IF (5′-GTTGAGGGTYGTTGAGTGTYGTAGGYGG-3′), and SPINT2IR (5′-CCAAATACCAAAACCCCCRTAAATCRCC-3′). The first PCR reaction (0.1 volume; SPINTF, SPINTR, annealing temperature: 58°C) was used in a second, nested reactions (SPINT2R, SPINT2IR and SPINT2IF, SPINT2R, annealing temperature: 58°C) The PCR conditions for both the first and second PCR were 95°C for 5 minutes, followed by 35 cycles of 45 seconds of denaturation at 95°C, 45 seconds of annealing at 58°C, and 45 seconds of extension at 72°C.

The methylation-specific primers were designed based on the sequencing data of the PCR-amplified bisulfite-modified cell line genomic DNA. The methylated alleles were amplified using SPINT2MSP-F (5′-CGGGCGTTTTTATATTGAAGGTTC-3′) and SPINT2MSP-R (5′-ACGCCACCAACCGTTAAAATCTCG-3′). The PCR cycling conditions for this reaction consisted of 5 minutes at 95°C followed by 30 cycles of 45 seconds of denaturation at 95°C, 45 seconds of annealing at 57°C, and 45 seconds of extension at 72°C. The unmethylated allele was amplified using primers Spint2USP-F (5′-GGTTGGGTGTTTTTATATTGAAGGTTT-3′) and SPINT2USP-R (5′ TCAACACCACCAACCATTAAAATCTCA 3′). The PCR cycling conditions were the same as for methylation-specific PCR.

**Mutation analysis of primary tumors and cell lines.** Intron-exon boundaries were determined by matching the cDNA sequence for _HAI-2/SPINT2_ with the working draft sequence of the human genome, the UCSC assembly 5 ([http://genome.ucsc.edu/](http://genome.ucsc.edu/)). Mutation screening was done by direct sequencing on an ABI 3730 automated sequencer. The sequences of the primers used along with the annealing temperature are available upon request.

**Immunoblotting.** Protein extraction and blotting was done essentially as described previously (33). Sections of frozen tissue were fractionated, weighed, and homogenized into 20-fold excess of extraction buffer [7 mol/L urea/10% glycerol/10 mmol/L Tris-HCl (pH 6.8)/1% SDS/5 mmol/L DTT/0.5 mmol/L phenylmethylsulfonyl fluoride (PMSF) and 1 mg/L of aprotinin, pepstatin, and leupeptin] with an electric homogenizer (Ultra-Turrax; IKA, Staufen, Germany). Extracts were quantified with the Bio-Rad detergent-compatible protein assay (Bio-Rad, Hemel Hempstead, Hertfordshire, United Kingdom). Protein lysates from cell line clones and mixed populations were prepared with NETS lysis buffer containing 3 mmol/L PMSF, 20 μg/mL aprotinin, and 10 μg/mL leupeptin (all chemicals from Sigma-Aldrich, Gillingham, Dorset, United Kingdom). Twenty micrograms of each extract were resolved on polyacrylamide gels. Proteins were transferred onto Immobilon P (Millipore, Bedford, MA) for 2 hours and probed with anti–HAI-2 goat antibody (R&D Systems, Minneapolis, MN) at 0.2 μg/mL. Signals were detected with horseradish peroxidase–conjugated antigoat antibody diluted 1/2,000 (DAKO, Ely, United Kingdom) and enhanced chemiluminescence (Amersham, Little Chalfont, Buckinghamshire, United Kingdom). After analysis, membranes were stained with India ink for standardization and quantification was done using a Bio-Rad imaging densitometer with Quantity One software.

**Plasmid constructs and colony formation assay.** The _HAI-2/SPINT2_ expression constructs were made by cloning the full-length human _HAI-2/SPINT2_ coding region from the SKRC 45 cell line, and a missense mutation found in the SKRC47 cell line, into the _Eco_R1_-Bam_HII sites of pCDNA3.1 vector (Invitrogen). Plasmid constructs were verified by sequencing. Ten micrograms of empty vector or expression vector were transfected, by calcium phosphate method, into 5 × 105 cells (either SKRC39, UMRC2, SKRC45, or COS-7). Forty-eight hours after transfection, cells were seeded in a serial dilution and maintained in DMEM and 10% fetal bovine serum supplemented with 1 mg/mL G418 (Life Technologies). Surviving colonies were stained with 0.4% crystal violet (Sigma) in 50% methanol, 14 to 21 days after initial seeding, and counted. Each transfection was carried out in triplicate. Additionally, replicate experiments were carried out to obtain further clones for expression analysis and further experiments.

_HAI-2/SPINT2_–conditioned media was prepared by transient transfection of COS-7 cells with wtSPINT2, mt_SPINT2_, or vector-only plasmids. Twenty-four hours later, the media was replaced with serum-free DMEM. Forty-eight hours after transfection, the media were cleared by centrifugation.

**Wound healing assay.** Three clones of SKRC39 and SKRC45 expressing high levels of exogenous wt_SPINT2_, mt_SPINT2_ (as verified by Western blotting), and control clones transfected with the empty vector were seeded at 5 × 104 in six-well plates, resulting in a confluent monolayer, and maintained in serum-free media. Each well of cells was scratched with the tip of a 200 μL pipette tip. Twenty-four hours following the scratch, the extent of “wound healing” was observed microscopically.

In a similar manner, untransfected SKRC39 cells were scratched and incubated in conditioned media from COS-7 cells transiently transfected (see above) with either wt_SPINT2_ or mt_SPINT2_. We also investigated whether treatment with signal transduction pathway inhibitors would modify the response to _HAI-2/SPINT2_ in the scratch would assay: MAPK inhibitors PD98059 (30 μmol/L; Calbiochem, San Diego, CA) and U0126 (20 μmol/L; Promega); phospholipase C-γ inhibitor U73122 (2 μmol/L; Calbiochem), and atypical protein kinase inhibitor GF109203X (1 μmol/L; Calbiochem). Serum-free media was supplemented with these inhibitors and SKRC39 cells were incubated for 24 hours then scratched as above and incubated for a further 24 hours.

**Anchorage-independent growth assay.** SKRC39, UMRC2, or SKRC45 clones (three of each) stably expressing wt_SPINT2_ or the empty vector control were suspended in DMEM 10% FCS agar. Cells were maintained by addition of 200 μL of DMEM 10% FCS, supplemented with 1 mg/mL G418 weekly. After 8 weeks of growth, a final count of colonies was done.

**Statistical analysis.** Paired _t_ tests and χ2 tests were carried out using SPSS 12.0 and statistical significance was taken at the 0.05 level.
#### Results
**Expression of HAI-2/SPINT2 protein in sporadic renal cell carcinoma**
**_HAI-2/SPINT2_ promoter region methylation status and mutation analysis in sporadic renal cell carcinoma****Restoration of _HAI-2/SPINT2_ expression reduces colony formation**
**Reexpression of _HAI-2/SPINT2_ inhibits anchorage-independent growth.**
**Reexpression of _HAI-2/SPINT2_ reduces cell motility.**
**Modulation of MET-downstream signal transduction and response to _HAI-2/SPINT2_ inactivation.**

#### Discussion

We have shown (_a_) tumor suppressor effects of HAI-2/SPINT2 in RCC cell lines, (_b_) frequent loss of expression and promoter region methylation of _HAI-2/SPINT2_ in RCC cell lines and primary tumors, and (_c_) a _HAI-2/SPINT2_ mutation with reduced tumor suppressor function in a RCC cell line. These findings implicate HAI-2/SPINT2 in the pathogenesis of clear cell and nonclear cell RCC, extend the role of disordered HGF/MET signaling in the pathogenesis of RCC, and provide a basis for developing novel therapeutic strategies.

Promoter methylation and transcriptional silencing of _HAI-2/SPINT2_ has been described previously in hepatocellular carcinoma cell lines and tumors (31). In addition, Yamauchi et al. (40) found that _HAI-2/SPINT2_ mRNA was abundantly expressed in normal kidney but was down-regulated in advanced stage RCC. In methylation profiles of RCC by us and others, only four (CASP8, RASSF1A, SLIT2, and TIMP3) of 17 tumor suppressor genes tested showed promoter methylation in >20% of RCC (24, 25). Thus, tumor suppressor gene methylation is not a generalized feature of RCC, and, relative to many other known tumor suppressors, methylation of _HAI-2/SPINT2_ in primary RCC tumors is frequent. We also investigated whether somatic mutations in _HAI-2/SPINT2_ were a common cause of _HAI-2/SPINT2_ inactivation in RCC. We did not detect mutations in 30 primary RCC tumors, but a single missense mutation (that inhibited _HAI-2/SPINT2_ tumor suppressor function) was detected in a RCC cell line. Thus, _HAI-2/SPINT2_ resembles other candidate RCC tumor suppressor genes, such as _RASSF1A_ and _NORE1A_, for which epigenetic inactivation is the major mechanism of inactivation (22, 41, 42).

HAI-2/SPINT2 is a Kunitz-type serine protease inhibitor that has a broad inhibitory spectrum against serine proteases, such as plasmin, trypsin, tissue, and plasma kallikreins and factor Xia (43, 44). HAI/SPINT2 was independently identified as placental bikunin. In studies of ovarian cancer, HAI-2/SPINT2/bikunin has been implicated as an inhibitor of tumor cell invasion and metastasis. In addition to inhibiting serine proteases, HAI-2/SPINT2 can also bind to high-affinity cell surface receptors (45) and down-regulate urokinase plasminogen activator (uPA) and its receptor (uPAR), probably by the MAP kinase pathway (46). Generation of plasmin from plasminogen by uPA can induce extracellular matrix degradation and promote tumor cell migration and metastasis. Hence, inactivation of HAI-2/SPINT2 might promote tumorigenesis by multiple mechanisms. Nevertheless, it seems likely that loss of HAI-2/SPINT2 inhibition of HGF/MET signaling would be implicated in promoting renal tumorigenesis. HGF activation of the MET signaling pathway induces profound effects on epithelial-cell motility, growth, and formation of branched tubules (47). During development, HGF is a potent growth factor for epithelial cells and has a critical role in regulating cellular motility (48–50). Dysregulation of the HGF/MET signaling pathway is frequent in human cancer and may result from a variety of mechanisms (47). Thus, up-regulation of HGF and MET expression can activate MET signaling by a paracrine effect. However, the most direct evidence for MET involvement in human cancer was provided by the finding of germ line activating _MET_ mutations in patients with HPRC1. Nevertheless, somatic _MET_ mutations are infrequent in renal cancer (3), so the finding of frequent _HAI-2/SPINT2_ inactivation in clear cell and papillary RCC provides an alternative mechanism by which dysregulated HGF/MET signaling can be implicated in the pathogenesis of RCC.

We found that restoration of _HAI-2/SPINT2_ expression had a variety of effects on RCC cell lines including suppression of _in vitro_ colony formation, inhibition of anchorage-independent growth, and reduced cell motility. Tumor formation and metastasis is a complex multistep process that requires the acquisition of a variety of properties (e.g., proliferation, invasion, angiogenesis, and antiapoptosis) that are associated with MET activation (51–54). We note that the biological effects of transfecting RCC cells with _HAI-2/SPINT2_ were most pronounced when endogenous HAI-2/SPINT2 was silenced, suggesting that the observed effects were specific consequences of _HAI-2/SPINT2_ overexpression. We have initiated preliminary investigations of the mechanisms of HAI-2/SPINT2 tumor suppression. Interestingly, we found that increased cell motility, associated with _HAI-2/SPINT2_ silencing, was abrogated by treatment with ERK/MAPK and phospholipase C-γ inhibitors, but not by inhibition of atypical PKC. The apparent partial inhibition of cell motility may be an indicator that HAI-2/SPINT2 inhibits a number of different signaling pathways (although we cannot exclude the possibility that the partial inhibition is an experimental artifact resulting from the relatively short half lives of the inhibitory compounds used). These findings provide a basis for further investigations into HAI-2/SPINT2 function and its dependence on specific downstream signaling pathways.

RCC is curable if detected at an early stage. However, up to 40% of patients with RCC present with locally advanced or metastatic disease that is difficult to treat with chemotherapy or radiotherapy. A major aim of human cancer genetics is to develop novel therapeutic agents based on a detailed knowledge of cancer molecular biology. Such agents could then be administered in individualized treatment regimens. Thus, response to treatment with an epidermal growth factor receptor (EGFR) kinase inhibitor (gefitinib) in lung cancer patients was associated with the presence of somatic EGFR mutations in the lung cancer (55, 56), Intriguingly, bikunin has been investigated as a treatment for ovarian carcinoma. Thus, overexpression of _HAI-2/SPINT2_ in an ovarian cancer cell line suppressed invasion and peritoneal carcinomatosis (30), and once-daily oral _HAI-2/SPINT2_ therapy reduced tumor load in a nude mouse model and in human ovarian cancer (57). In addition, a combination of _HAI-2/SPINT2_ and paclitaxel produced more profound effects on tumor growth (58). We note that conditioned media from HAI-2/SPINT2–expressing cells inhibited cell motility. Thus, our findings suggest that administration of HAI-2/SPINT2, and/or inhibitors of MET signaling (59), might provide novel therapeutic approaches to treating advanced RCC with HAI-2/SPINT2 inactivation.


## Extra Research
## Misc
![pBluescript_II_SK(+).png](/img/user/images/pBluescript_II_SK(+).png)
![pcDNA3.1(-).png](/img/user/images/pcDNA3.1(-).png)
![Pasted image 20251109213326.png](/img/user/images/Pasted%20image%2020251109213326.png)
![Pasted image 20251109213332.png](/img/user/images/Pasted%20image%2020251109213332.png)
![Pasted image 20251113002319.png](/img/user/images/Pasted%20image%2020251113002319.png)
![Pasted image 20251113002348.png](/img/user/images/Pasted%20image%2020251113002348.png)
![Pasted image 20251109213338.png](/img/user/images/Pasted%20image%2020251109213338.png)
![Pasted image 20251109213401.png](/img/user/images/Pasted%20image%2020251109213401.png)
![Pasted image 20251109213525.png](/img/user/images/Pasted%20image%2020251109213525.png)
![Pasted image 20251113002401.png](/img/user/images/Pasted%20image%2020251113002401.png)
![Pasted image 20251113002427.png](/img/user/images/Pasted%20image%2020251113002427.png)
![Pasted image 20251109213621.png](/img/user/images/Pasted%20image%2020251109213621.png)
![Pasted image 20251109215235.png](/img/user/images/Pasted%20image%2020251109215235.png)
![Pasted image 20251109220206.png](/img/user/images/Pasted%20image%2020251109220206.png)
![Pasted image 20251109220215.png](/img/user/images/Pasted%20image%2020251109220215.png)
![Pasted image 20251109222044.png](/img/user/images/Pasted%20image%2020251109222044.png)
![Pasted image 20251109215943.png](/img/user/images/Pasted%20image%2020251109215943.png)
![Pasted image 20251109220003.png](/img/user/images/Pasted%20image%2020251109220003.png)
![Pasted image 20251109220012.png](/img/user/images/Pasted%20image%2020251109220012.png)
![Pasted image 20251109220026.png](/img/user/images/Pasted%20image%2020251109220026.png)
![Pasted image 20251109220326.png](/img/user/images/Pasted%20image%2020251109220326.png)
![Pasted image 20251109220333.png](/img/user/images/Pasted%20image%2020251109220333.png)
![Pasted image 20251109220612.png](/img/user/images/Pasted%20image%2020251109220612.png)
![Pasted image 20251109220729.png](/img/user/images/Pasted%20image%2020251109220729.png)
![Pasted image 20251109221116.png](/img/user/images/Pasted%20image%2020251109221116.png)
![Pasted image 20251109221356.png](/img/user/images/Pasted%20image%2020251109221356.png)
![Pasted image 20251109221428.png](/img/user/images/Pasted%20image%2020251109221428.png)
![Pasted image 20251109221820.png](/img/user/images/Pasted%20image%2020251109221820.png)
![Pasted image 20251109222235.png](/img/user/images/Pasted%20image%2020251109222235.png)
![Pasted image 20251109222300.png](/img/user/images/Pasted%20image%2020251109222300.png)
![Pasted image 20251109223041.png](/img/user/images/Pasted%20image%2020251109223041.png)
![Pasted image 20251109223046.png](/img/user/images/Pasted%20image%2020251109223046.png)
![Pasted image 20251109223117.png](/img/user/images/Pasted%20image%2020251109223117.png)
![Pasted image 20251109223122.png](/img/user/images/Pasted%20image%2020251109223122.png)
![Pasted image 20251109223134.png](/img/user/images/Pasted%20image%2020251109223134.png)

![Pasted image 20251113001917.png](/img/user/images/Pasted%20image%2020251113001917.png)
![Pasted image 20251113001937.png](/img/user/images/Pasted%20image%2020251113001937.png)
![Pasted image 20251113001956.png](/img/user/images/Pasted%20image%2020251113001956.png)
![Pasted image 20251113002004.png](/img/user/images/Pasted%20image%2020251113002004.png)
![Pasted image 20251113002015.png](/img/user/images/Pasted%20image%2020251113002015.png)
![Pasted image 20251113002021.png](/img/user/images/Pasted%20image%2020251113002021.png)
## Stuff I Missed
![Pasted image 20251109213413.png](/img/user/images/Pasted%20image%2020251109213413.png)
![Pasted image 20251109213423.png](/img/user/images/Pasted%20image%2020251109213423.png)
![Pasted image 20251109214106.png](/img/user/images/Pasted%20image%2020251109214106.png)
![Pasted image 20251109214222.png](/img/user/images/Pasted%20image%2020251109214222.png)

![Pasted image 20251109214703.png](/img/user/images/Pasted%20image%2020251109214703.png)
![Pasted image 20251109214737.png](/img/user/images/Pasted%20image%2020251109214737.png)
![Pasted image 20251109214757.png](/img/user/images/Pasted%20image%2020251109214757.png)
![Pasted image 20251109215257.png](/img/user/images/Pasted%20image%2020251109215257.png)
![Pasted image 20251109215303.png](/img/user/images/Pasted%20image%2020251109215303.png)
![Pasted image 20251109215309.png](/img/user/images/Pasted%20image%2020251109215309.png)
![Pasted image 20251109222528.png](/img/user/images/Pasted%20image%2020251109222528.png)
![Pasted image 20251109222532.png](/img/user/images/Pasted%20image%2020251109222532.png)



## Definitions 

Complimentary DNA (cDNA): DNA synthesised from a mRNA template catalysed by the retroviral enzyme reverse transcriptase.

Oligonucleotide: a short nucleic acid polymer, in vivo these can be generated by enzymes such as primase. Commercially, they are generated in vitro by polymerising individual nucleotide precursors.


Open Reading Frame (ORF): the portion of a gene that comprises the coding base pair triplets for a protein and nothing else (all introns, 5’ and 3’ non-coding regions have been removed). 

Primase: Enzymes whose continual activity is required at the DNA replication fork. They catalyze the synthesis of short RNA molecules used as primers for DNA polymerases to extend new daughter DNA strands from.

Reverse Transcriptase: An enzyme used to generate DNA from an RNA template. The enzyme is encoded by reverse-transcribing RNA viruses (e.g. retroviruses) to convert their RNA genomes into DNA  

## Appendix  
1.1: Preparation of calcium competent E.coli.  
Day 1  
Set up an overnight culture of E.coli  
Day 2  
1. Inoculate 20 ml of Luria broth with 0.75 ml of the O/N E.coli culture.  
2. Incubate, with aeration, for 90-100 minutes at 37°C (to an OD~O.6).  
3. Spin down the cells at 10 K for 30 seconds.  
4. Wash the cells in 5 ml of 5 mM NaCl and spin down at 10K for 5 minutes at 4°C.  
5. Resuspend the cells in 2 ml of 100 mM CaCl2 and incubate on ice for at least 20  
minutes.  
6. Spin the cells down at 10 K for 5 minutes at 4°C.  
7. Resuspend the cells in 1 ml of 100 mM CaC12


1.2a: Historic mini-prep protocol Solutions  
Solution GTE:               50 mM glucose, 25 mM Tris/HCl (pH 8.0), 10 mM EDTA  
Solution SDS/NaOH: 0.2 M NAOH, 1 % (w/v) SDS (sodium dodecyl sulphate)  
Solution KAc:                5 M Potassium Acetate (60 ml), Glacial acetic acid (11.5 ml),  
H2O (28.5 ml).  

1.2b: Ion exchange column miniprep protocol (Qiagen)

1.3: pcDNA3.1 plasmid map

1.4: pBluescript plasmid map

1.5 ThermoScientific 1Kb DNA Ladder

1.6 p35/10 BAC  


## Assessment  
This is a single practical which extends over 6 sessions.  
During that time you will carry out several experiments each of which will continue on  
from the one before, forming a natural progression.  
You will produce a number of results (gel images, Graphs etc.) These should be kept  
and submitted to CANVAS as instructed in this handbook and in the class lectures  
(marks will be given for these, so do not lose them).  
Answer the questions that accompany the methodology; submit them to  
CANVAS in as instructed.  
You should consider these as short answer questions, essays are not required but most  
will need more than a single sentence to answer!  
There will also be a written test at week six.  
The week 6 test is worth 50% of your final mark of this cloning practical, the  
remaining 50% will be assessed on your answers to the questions in the lab book  
and data generated (Gel Images, graphs etc.).  
A formal write-up is not required.  
At the end of each weekly section are instructions on what you should prepare and  
submit before the following week (and the maximum number of marks available  
for each component).  
OBJECTIVES  
The purpose of this module is to introduce you to the basic techniques of molecular  
biology related to the manipulation of recombinant DNA and the use of microbial host  
cells. The techniques you will use are fundamental to modern molecular biology:  
• Restriction endonuclease analysis of plasmid DNA  
• Creating recombinant DNA by ligation  
• Transformation of bacterial host cells with plasmid DNA  
• Rapid isolation of recombinant plasmids from bacterial host cells  
An interesting gene (SPINT2) has been cloned into a basic cloning plasmid. To  
enable us to determine what the gene product (protein) does we need to sub-clone  
it into an expression plasmid.  
This will be done using a number of molecular biology techniques over the next 5  
weeks!

3  
EXPERIMENTAL PLAN  
Week 1  
- Restriction Digest cloning plasmid (pBluescript) containing SPINT2.  
- Restriction Digest mammalian expression plasmid (pCDNA3.1).  
- Ligate Restriction digested SPINT2 with expression plasmid (pCDNA  
3.1).  
Week 2  
- Transformation of competent E. Coli with product of ligation and control  
plasmids.  
- Plate transformed E. Coli onto selective (ampicillin) agar plates and  
incubate 16hrs  
- Calibrate the size of DNA fragments run on an agarose gel (dry).  
- Introduction to data analysis (Avogadro’s rule).  
Week 3  
- Score successful transformants (from week 2)  
- Gel electrophotresis of restriction digest (from week 1).  
- Calibrate the size of your restriction fragments following gel  
electrophoresis.  
- Pick Ampr E. Coli colonies from agar plates (from week 2) and incubate  
16hrs in luria broth containing ampicillin.  
Week 4  
- Plasmid mini-preps from E. Coli colonies picked in week 3  
- Restriction enzyme digest of plasmid mini-preps  
- Calibrate the size of your restriction fragments following gel  
electrophoresis.  
Week 5  
- Gel electrophoresis of mini-prep restriction digests (from week 4)  
- Calibrate the size of your restriction fragments following gel  
electrophoresis.  
- Have you successfully sub-cloned a gene from a cloning plasmid to an  
expression plasmid??  
Week 6  
- Test!  
By the end of the practical, you should be familiar with the methods needed for these  
experiments, and understand how they can be applied in other experiments.  
In these notes, a number of questions, which are relevant to particular steps in the  
procedure, have been included. You should answer these while working through the  
practicals. We recognize that practicals do not always "work" and so you will be  
assessed more on your understanding of basic principles than on the actual results of  
your experiments.

3  
EXPERIMENTAL PLAN  
Week 1  
- Restriction Digest cloning plasmid (pBluescript) containing SPINT2.  
- Restriction Digest mammalian expression plasmid (pCDNA3.1).  
- Ligate Restriction digested SPINT2 with expression plasmid (pCDNA  
3.1).  
Week 2  
- Transformation of competent E. Coli with product of ligation and control  
plasmids.  
- Plate transformed E. Coli onto selective (ampicillin) agar plates and  
incubate 16hrs  
- Calibrate the size of DNA fragments run on an agarose gel (dry).  
- Introduction to data analysis (Avogadro’s rule).  
Week 3  
- Score successful transformants (from week 2)  
- Gel electrophotresis of restriction digest (from week 1).  
- Calibrate the size of your restriction fragments following gel  
electrophoresis.  
- Pick Ampr E. Coli colonies from agar plates (from week 2) and incubate  
16hrs in luria broth containing ampicillin.  
Week 4  
- Plasmid mini-preps from E. Coli colonies picked in week 3  
- Restriction enzyme digest of plasmid mini-preps  
- Calibrate the size of your restriction fragments following gel  
electrophoresis.  
Week 5  
- Gel electrophoresis of mini-prep restriction digests (from week 4)  
- Calibrate the size of your restriction fragments following gel  
electrophoresis.  
- Have you successfully sub-cloned a gene from a cloning plasmid to an  
expression plasmid??  
Week 6  
- Test!  
By the end of the practical, you should be familiar with the methods needed for these  
experiments, and understand how they can be applied in other experiments.  
In these notes, a number of questions, which are relevant to particular steps in the  
procedure, have been included. You should answer these while working through the  
practicals. We recognize that practicals do not always "work" and so you will be  
assessed more on your understanding of basic principles than on the actual results of  
your experiments.

4  
Outline of sub-  
cloning  
WEEK 1  
Gene Sub-cloning  
Step 1  
Restriction digest of plasmids to prepare gene (insert) and expression vector for  
ligation.  
Restriction enzymes cut DNA at specific recognition sites determined by the base  
sequence. They are routinely kept as glycerol (antifreeze) stocks in the freezer until  
needed when they are kept on ice. They are usually named after the organism from  
which they were isolated (e.g. EcoRI). You must avoid contact of skin with any  
implements that may touch DNA as your skin contains nucleases that will non-  
specifically degrade DNA. Often we wear disposable gloves for this purpose.  
Restriction enzymes are measured in units of activity where 1 unit will digest 1 g of  
DNA to completion in 1 hour. Normally we add a concentrated buffer to DNA, suitably  
diluted in sterile distilled water, and finally add enzyme such that the enzyme works in  
the final strength buffer solution. Most restriction enzymes work at 370C, digestion  
proceeds for one to a few hours.

5  
Method  
(To be done individually)  
1) You are provided with two microfuge tubes:  
A) 2ug pBluescript-SPINT2 (volume: 4ul, conc. 500ng/ul)  
B) 2ug pCDNA3.1 (volume: 4ul, conc. 500ng/ul)  
2) To each tube add 12ul H2O  
3) Add restriction enzymes and buffer to each of these tubes:  
1ul Xho 1  
1ul Hind III  
2ul 10X restriction enzyme buffer  
The final volume of each reaction is 20ul.  
(1ul of each enzyme contains 10 units).  
Volumes in l  
DNA WATER 10x BUFFER ENZYME  
4ul 12ul 2ul 1ul Xho1  
1ul HindIII  
4) Incubate enzymes for 2hrs at 37OC  
We will run the products of these digests by electrophoresis on an agarose gel in  
WEEK 3.  
Q1: What is the recognition sequence of the enzymes Xho1 and HindIII  
Q2: Would the restriction reaction work at a) 4OC or b) 95OC  
Due to time constraints we will continue with purified products of this digest that have  
been prepared for you!

6  
Step 2: Ligation  
Introduction  
DNA Ligase was discovered in 1967. This enzyme catalyzes the formation of a  
phosphodiester bond between the 3’-hydroxyl group at the end of one double stranded  
DNA chain and the 5’-phosphoryl group at the end of another double stranded DNA  
chain. In eukaryotes DNA ligase is used during DNA replication to join Okazaki  
fragments on the lagging strand. The isolation of DNA ligase was an essential point in  
the development of recombinant DNA technology.

6  
Step 2: Ligation  
Introduction  
DNA Ligase was discovered in 1967. This enzyme catalyzes the formation of a  
phosphodiester bond between the 3’-hydroxyl group at the end of one double stranded  
DNA chain and the 5’-phosphoryl group at the end of another double stranded DNA  
chain. In eukaryotes DNA ligase is used during DNA replication to join Okazaki  
fragments on the lagging strand. The isolation of DNA ligase was an essential point in  
the development of recombinant DNA technology.

7  
DNA Ligase reaction.

8  
Method  
1) You are provided with two microfuge tubes:  
C) 4ul pCDNA3.1 pre-cut with the  
restriction enzymes Xho1/HindIII and  
purified @25ngul  
D) 4ul SPINT2 gene cut out of pBluescript  
with the restriction enzymes  
XHO1/HindIII and purified @12ngul  
2) Add the contents of tube C to tube D.  
3) Add 9ul H2O  
4) Add:  
a) 1ul T4 DNA Ligase (5 units)  
b) 2ul Ligase buffer  
5) Incubate the reaction at 4OC for 16hrs  
To increase the chances of ligating one copy of the gene (SPINT2) into one copy of the  
expression vector (pcDNA3.1) we add more gene than vector. How much we add is not  
measured in ug of DNA but is measured in the number of sticky ends of each molecule  
in the reaction. It is common to use three times as many ends of gene compared to  
vector.  
This is calculated using the following calculation:  
ATG$ Xho1%  
HindIII% TGA$SPINT2'~800 bp

9  
For a 1:1 ratio of molecule ends you would need 15 ng of insert for every 100ng of  
vector. You are ligating 3 times the amount of insert ends than vector (3:1) therefore  
need to add 45ng of insert to 100ng of vector.  
Formative assessment Questions:  
Q1: What is the recognition sequence of the enzymes Xho1 and HindIII  
Q2: Would the restriction reaction work at a) 4OC or b) 95OC  
(Formative assessment questions do not need to be handed in as homework and  
carry no assessment weight. BUT, considering them carefully and formulating  
appropriate answers will benefit you in many ways!)  
Week 2  
TRANSFORMATION OF E. COLI  
Transformation is the uptake of exogenous DNA from outside the cell. Many bacteria  
are naturally “competent’, that is they are normally ready to take up DNA under normal  
growth conditions. Other bacteria, like E. coli require special treatments. In this case,  
the standard protocols use temperature and calcium shock treatments.  
Following a ligation reaction only a small proportion of inserts and vectors successfully  
ligate. It is necessary to amplify these by transforming bacteria and growing successful  
transformants. Individual transformed bacteria will grow into a single colony of cells,  
each individual cell in a colony will contain identical copies of the transformed plasmid.  
By this process the gene (insert) ligated into the vector (plasmid) is cloned.  
You are provided with an overnight culture of competent E. coli. You will attempt  
transformation with the pcDNA3.1-SPINT2 ligations you prepared in week 1. As  
controls for this experiment you will also transform empty (circular) pcDNA3.1 and  
H2O. You will also transform the large p35/10 plasmid (49.5kb (see appendix 1.6)).  
pcDNA3.1 and p35/10 carry the gene that encodes resistance to ampicillin.

10  
Method  
(To be done individualy)  
1) Preparation of cells for transformation  
Pay attention to the precise instructions given by the demonstrators. The supplied  
plasmids carry an ampicillin resistance gene but differ in their molecular mass.  
You are provided with a 1 ml culture of E. coli cells re-suspended in ice cold 100 mM  
CaCl2. This calcium treatment renders the cells competent for the uptake of exogenous  
DNA (See appendix 1.1 for details of competent cell preparation).  
2) Prepare four 1.5ml microfuge (labeled A-D) tubes by adding 0.1 ml of the cells then  
add:  
a) 10ul of the pcDNA3.1-SPINT2 ligation (from week 1)  
b) 5ul pcDNA3.1 (Conc. 10ng/ul)  
c) 5ul p35/10 (Conc. 10ng/ul)  
d) 5ul H2O  
Only add one thing per tube! And label them with their content and  
your name.  
Gently mix.  
3) Incubate the cells with the DNA on ice for 30 minutes, then place the tubes at 42°C  
for 1.5-2 minutes. Immediately place the tubes on ice for a minimum of 2 mins.  
4) Remove from ice and add 900ul of Luria broth to each tube, incubate at 37°C for  
40mins-1hr.  
5) After this period of incubation without antibiotic selection plate 100ul of sample b,  
c and d onto an agar plate containing ampicillin (@50ug/ml).  
Spin the tube containing your ligation (A) @ 2000rpm for 4 min. gently pour the  
content into the chloros pot leaving a small volume (aprox 100ul) re-suspend the pellet.  
Plate all of this onto an agar plate containing ampicillin (@50ug/ml).  
6) The plates will be incubated for you overnight at 37°C, then stored in the ‘fridge  
until next week.  
Formative assessment Questions:  
Q3: Why is it necessary to incubate the transformed bacterial cells in  
media that does not contain any antibiotics before plating them onto growth  
media containing the antibiotic ampicillin?  
(Formative assessment questions do not need to be handed in as homework and  
carry no assessment weight. BUT, they are good for you!)

11  
Assume:  
- Plasmids are supplied at 10 μg DNA/ml.  
- Cells at 5 x 108/ml.  
- 1.5 kilobases of duplex DNA corresponds to approximately 1 x 106 daltons, molecular  
weight.  
- Avogadro’s number is 6.02 x 1023 /mol  
Assessed Question  
Q4: Calculate the number of molecules of pcDNA3.1 in 50ng (Show your  
workings) (10 Marks).  
Analysis of agarose gel electrophoresis patterns  
Below is a picture of an electrophoresis gel.  
Lane 1 contains HindIII-digested lambda DNA as a size marker.  
Lane 2 contains pCDNA3.1 digested with EcoR1.  
Lane 3 contains pCDNA3.1 digested with Sal1.  
Lane 4 contains pCDNA3.1 digested with Nco1.  
Each band in the lambda ladder is of a known size, as given in the picture to the right  
of the gel image. Compare the diagram of the Lambda-HindIII ladder with the gel  
picture and identify each of the bands. You now know the size of each one. Measure  
the distance from the well (top of the gel image) to each band. You can now plot a  
graph of the distance each band has traveled against its size. Plot the distance migrated  
against the log of the size for each one of the lambda ladder marker bands.  
Measure the distances that the bands in lanes 2, 3 and 4 have migrated from the wells.  
Tabulate these results. Note that the larger the molecular weight of the DNA fragment,  
the more slowly it will move through the gel.  
Now use your graph to estimate the sizes of the restriction fragments in each lane by  
measuring the distance each has traveled.

12  
To be handed in on week 3:  
- Graph generated from electrophoresis measurements and estimations of  
fragment sizes (20 marks)  
- Answers to Q4 (10 marks)

13  
Week 3  
Gel electrophoresis of restriction digests (from Week 1)  
At the beginning of this practical you digested the cloning plasmid pBluescript, that  
had the gene SPINT2 cloned into its multiple cloning site (at the BamH1 site) and the  
mammalian expression vector pCDNA3.1 with the restriction enzymes HindIII and  
Xho1. The purpose of this was to isolate the SPINT2 gene and pCDNA3.1 vector with  
complimentary cohesive (sticky) ends. Once these have been purified they can be  
ligated together (as you did with the fragments provided in week 1).  
Today we are going to run these digests on an agarose gel by electrophoresis. This is  
the first step in purifying the products in preparation for ligation.  
DNA is negatively charged and is loaded into the slots in the gel in the presence of a  
loading buffer containing a substance to increase density (like sucrose) and a blue  
tracker dye. The gel is submerged in a buffer (TBE). Application of the electric field  
encourages the DNA to migrate towards the anode. After a suitable period of time (an  
hour to overnight) the gel is irradiated with ultraviolet light (mutagen – take care) to  
visualise the DNA. The DNA is visible because the gel contains ethidium bromide  
(mutagen) or a safer alternative.  
Method  
(To be done individually)  
1) Collect your digest reactions.  
2) Add 4ul of 6X bromophenol blue loading buffer to each of the reaction tubes.  
3) 1% agarose gels have been prepared in advance. You will load 3 lanes:  
-1: 10ul DNA ladder  
-2: 20ul pBluescript-SPINT2 Xho1/HindIII  
-3: 20ul pCDNA3.1 Xho1/HindIII  
Make sure to note which gel, and the position on that gel, that your  
samples have been loaded!  
4) The gels will be run @ 100v for 1 hr (or until sufficient separation has  
occurred).  
5) Visualise you digest products under UV (Do not look directly at the UV  
source-use Perspex shield!)  
6) We will provide you with electronic versions of your results. Estimate the size  
of the fragments (a representation of the DNA ladder (Norgen highranger) can  
be found in appendix 1.5). Generate a labeled figure (a calibration curve is  
not required). This analysis will be part of the assessment for this  
practical.  
The distances run on a gel are specific to the particular gel and running time. Other  
unknown linear DNA molecules can be accurately sized but the sizes of circular  
molecules can only be approximately estimated as their topological form (such as the  
degree of supercoiling) will also influence mobility in the gel.

14  
Formative assessment Questions:  
Q5: what do the bands you see represent?  
Q6: What does the TBE do?  
Q7: How does ethidium bromide work?  
(Formative assessment questions do not need to be handed in as homework and  
carry no assessment weight. BUT, when you spend time discovering the answers  
you bring joy to the lab!)  
Scoring successful transformants  
1) Collect your Agar plates  
Score the numbers of colonies occurring on each plate.  
Assessed Questions  
Q8: Express the transformation frequency for each batch as:  
- transformants per μg of DNA.  
- transforming molecules of DNA as a fraction of the number of molecules of input  
DNA.  
(20 Marks)  
Q9:  
a) Which plate has the most/ least colonies?  
b) Why is this?/ is this what you expected?  
c) Does the number of colonies obtained on your ligation plate correlate well to  
the amount of DNA used in the transformation?  
(10 Marks)

15  
Pick and culture transformant colonies  
Some (by no means all) of the transformed colonies on your ligation plate should  
contain re-circularised pCDNA3.1 now containing the gene SPINT2 ligated between  
the HindIII and Xho1 restriction sites. Before we can purify this (to use in experiments  
to determine the function of the gene) we need to identify a colony that contains a  
successfully ligated clone. Individual colonies will be picked and grown in liquid  
culture to produce sufficient starting material to isolate purified plasmid.  
At this point in the process research scientists will pick many (20-50) individual  
colonies to increase the likelihood of finding a successful clone.  
To increase our chances of success each student (not pairs) should pick 1 colony  
from their control plate (pCDNA3.1) and 1 colony from the ligation plate.  
Pipette 1.2m1 of L-broth (containing ampicillin @ 50ug/ml) into:  
1) A 1.5ml microfuge tube labeled with your name and “pCDNA3.1”.  
2) A 1.5ml microfuge tube labeled with your name and “ligation”.  
Using a sterile loop, gently scrape up a single colony from the pCDNA3.1 plate. Place  
the colony into the respective tube (1)  
Repeat with a colony from the ligation plate and tube 2.  
Sterilise the loop between inoculations!  
Put the tubes into the rack provided. The tubes will be placed into a rotating (orbital)  
incubator at 37°C for 16hrs then stored @ 4OC.  
Next week plasmid clones will be isolated from these colonies and restriction digests  
will be preformed to determine if you have successfully sub-cloned SPINT2 into the  
expression vector pCDNA3.1.  
To be handed in on week 4:  
- Determination of your restriction fragment sizes (20 marks)  
- Answers to Q8 (20 marks) and Q9 (10 marks)

16  
Week 4  
Isolation of plasmid DNA from a culture of E. Coli cells (minipreps)  
Today you will prepare plasmid DNA from your bacterial cultures and digest your  
product with HindIII and Xho1 restriction enzymes to determine successes.  
Follow the instructions carefully; the minipreps will not work unless you do so. It may  
help to tick off each point as it is completed.  
Preparation of plasmid DNA from overnight cultures (minipreps)  
Outlined below (for your reference) is a manual method for plasmid miniprep. Today,  
you will use a commercial ion-exchange method (the method can be found IN THE  
APPENDIX and on CANVAS and will be described in the lab session).  
1) Collect your 2 cultures (from week 3). Make sure that your tubes are labeled with  
your name. Centrifuge the tube for 1 min @ Max speed to pellet the cells. Pour off the  
supernatant into disinfectant (chloros) and drain the residual supernatant by inverting  
the tube onto a tissue. The bacterial pellet should be as dry as possible. (If you do not  
do this carefully the plasmid may not be isolated).  
2) Re-suspend the pellet by adding 100l of ice-cold solution GTE to the tube and  
ensure that the bacteria are fully suspended by vortexing thoroughly. This may take up  
to a minute. Ensure that ALL the cells have been resuspended by holding the  
suspension up to the light and looking for lumps. The bacterial pellet must be  
completely dispersed.  
3) Add 200l of solution SDS/NaOH at room temperature. Close the tube tightly, and  
mix the contents by inverting the tube rapidly five times. Do not vortex. Stand the tube  
on ice for about 5 minutes. The lysate should now be very viscous.  
4) Add 150 l of ice-cold solution of potassium acetate. Close the tube and mix it gently  
for 15 seconds to disperse the solution through the viscous bacterial lysate. Stand the  
tube on ice for 5 minutes. A white precipitate should appear.  
5) Centrifuge for 8 minutes in a microcentrifuge at full speed to pellet the precipitate.  
Transfer the supernatant to a fresh labeled tube taking care not to pipette any of the  
white precipitate. Discard the tube containing the precipitate.  
6) Add an equal volume of isopropanol (about 400l) to the supernatant in the fresh  
tube and invert it quickly about 12 times. Stand at room temperature for a couple of  
minutes before centrifuging at full speed for 5 minutes to pellet the nucleic acids which  
should have precipitated.  
7) Remove the supernatant without disturbing the nucleic acid pellet (be careful, pellets  
are very small and easily detached). Wick off as much of the isopropanol as you can  
onto a paper towel.  
8) Wash the pellet with 200l of 95% ethanol, flicking the tube to disturb the pellet.

17  
9) Centrifuge for 5 minutes at full speed and remove ALL of the supernatant. (Be very  
careful the pellet is both very small and very mobile. Do not throw it away by accident)  
Stand the tube opened to the air for about 10 minutes to remove/evaporate all of the  
a1cohol. Make sure that you have removed all of the a1cohol or the digest will not  
work.  
10) Dissolve the pellet in 20 l of H2O. The plasmid preps can now be store at -20°C  
until needed.  
Se appendix 1.2 for solutions used in this mini-prep protocol  
Formative assessment Questions:  
10: What is the function of the GTE?  
11: What is the function of the SDS/NaOH?  
12: What is the function of the potassium acetate?  
13: What is the function of the isopropanol?  
(Formative assessment questions do not need to be handed in as homework and  
carry no assessment weight. BUT, answering these questions will make you a  
better person and better prepared for the test!)  
Using a restriction endonuclease, you will restrict the plasmid nucleic acid which you  
have isolated. This should free the insert. The plasmid and insert can be separated by  
running the products of restriction digest on an agarose electrophoresis gel.  
Restriction endonuclease digestion of plasmid mini-preps.  
11) Label three microtubes and set up digests as follows (tube 1 is a control). Prepare  
the DNA and H2O. add 10X buffer.  
Tube Miniprep/DNA Buffer HindIII Xho1 H2O  
12. 2l 1l 1l 16l  
13. Ligation: 10l 2l 1l 1l 6l  
14. pCDNA3.1 ct: 10l 2l 1l 1l 6l  
15) Incubate the reactions @ 37OC for 2 hrs then freeze to stop the reaction.  
Nothing needs to be handed in on week 5, but one day you will look  
back at this week and be thankful for the time you spent on the  
formative assessments (maybe not today, but soon...)

18  
Week 5  
Gel electrophoresis of cloned DNA restriction digests  
Today you will run your digested mini-preps on an agarose gel by electrophoresis (see  
week 3 for details of electrophoresis) to determine if you have:  
a) Successfully amplified and purified plasmid DNA by the quick miniprep method  
(week 4).  
b) Successfully sub-cloned the SPINT2 gene from the cloning plasmid pBluescript  
into the mammalian expression plasmid pCDNA3.1.  
1) Collect your digest reactions.  
2) Add 4ul of 6X bromophenol blue loading buffer to each of the reaction tubes.  
3) 1% agarose gels have been prepared in advance. You will load 4 lanes:  
-1: 10ul DNA ladder  
-2: 20ul H2O control  
-3: 20ul ligation Xho1/HindIII  
-4: 20ul pCDNA3.1 Xho1/HindIII  
Make sure to note which gel, and the position on that gel, that your  
samples have been loaded!  
4) The gels will be run @ 100v for 1 hr.  
5) Visualise you digest products under UV (Do not look directly at the UV  
source-use Perspex shield!)  
6) We will provide you with electronic versions of your results. Determine the  
size of the fragments as done in week 2.  
7) Consult a demonstrator about your gel results.  
8) Have you successfully sub-cloned the gene into pCDNA3.1?  
Assessed Questions  
Q14: describe what you see in each of the three digestion lanes, what do  
they represent? Was your sub-cloning experiment successful?  
To be handed in on week 6:  
- Answer to Q14, including a fully labeled figure (a calibration graph is  
NOT required) (30 marks).  
Week 6  
Test!  
The test will last for 1 hour 30 min.


## SPINT2

Available structures
PDB	Ortholog search: PDBe RCSB
List of PDB id codes

4U32
Identifiers
Aliases	SPINT2, DIAR3, HAI-2, HAI2, Kop, PB, serine peptidase inhibitor, Kunitz type, 2, serine peptidase inhibitor, Kunitz type 2
External IDs	OMIM: 605124; MGI: 1338031; HomoloGene: 7955; GeneCards: SPINT2; OMA:SPINT2 - orthologs
Gene location (Human)
Chromosome 19 (human)
Chr.	Chromosome 19 (human)[1]
Chromosome 19 (human)
Genomic location for SPINT2
Genomic location for SPINT2
Band	19q13.2	Start	38,244,035 bp[1]
End	38,292,615 bp[1]
Gene location (Mouse)
Chromosome 7 (mouse)
Chr.	Chromosome 7 (mouse)[2]
Chromosome 7 (mouse)
Genomic location for SPINT2
Genomic location for SPINT2
Band	7|7 B1	Start	28,955,748 bp[2]
End	28,981,337 bp[2]
RNA expression pattern
Bgee	
Human	Mouse (ortholog)
Top expressed in

    beta cell

    mucosa of transverse colon

    parotid gland

    pituitary gland

    anterior pituitary

    olfactory zone of nasal mucosa

    left lobe of thyroid gland

    right lobe of thyroid gland

    right uterine tube

    right testis

	
Top expressed in

    left colon

    submandibular gland

    intestinal villus

    crypt of lieberkuhn of small intestine

    pyloric antrum

    duodenum

    parotid gland

    ileum

    right kidney

    epithelium of small intestine

More reference expression data
BioGPS	
More reference expression data
Gene ontology
Molecular function	

    peptidase inhibitor activity
    endopeptidase inhibitor activity
    serine-type endopeptidase inhibitor activity
    protein binding

Cellular component	

    cytoplasm
    integral component of membrane
    extracellular region
    membrane
    plasma membrane

Biological process	

    negative regulation of cell motility
    negative regulation of peptidase activity
    negative regulation of cell-cell adhesion
    establishment or maintenance of cell polarity
    negative regulation of neural precursor cell proliferation
    neural tube closure
    negative regulation of endopeptidase activity
    epithelial cell morphogenesis involved in placental branching
    cellular response to BMP stimulus
    basement membrane organization

Sources:Amigo / QuickGO
Orthologs
Species	Human	Mouse
Entrez	

10653
	

20733
Ensembl	

ENSG00000167642
	

ENSMUSG00000074227
UniProt	

O43291
	

Q9WU03
RefSeq (mRNA)	

NM_021102
NM_001166103
	

NM_001082548
NM_011464
RefSeq (protein)	

NP_001159575
NP_066925
	

NP_001076017
NP_035594
Location (UCSC)	Chr 19: 38.24 – 38.29 Mb	Chr 7: 28.96 – 28.98 Mb
PubMed search	[3]	[4]



<center><sub>Done I Hope :(</sub></center>