---
{"dg-publish":true,"permalink":"/uni/2/molecular-biosciences/5-bc-005-molecular-biosciences-practical-techniques/molecular-biosciences-practical-techniques/","tags":["Tagless"],"noteIcon":""}
---

<style id="Force_Custom_Fonts" type="text/css">@font-face{font-style:normal;font-family:"Merriweather";src:local("Merriweather")}@font-face{font-style:bolder;font-family:"Merriweather";src:local("Merriweather")}@font-face{font-style:normal;font-family:"Merriweather";src:local("Merriweather");unicode-range:U+0-FF,U+2E80-9FFF,U+F900-FAFF,U+FE30-FE4F,U+20000-2FA1F}@font-face{font-style:bolder;font-family:"Merriweather";src:local("Merriweather");unicode-range:U+0-FF,U+2E80-9FFF,U+F900-FAFF,U+FE30-FE4F,U+20000-2FA1F}@font-face{font-style:normal;font-family:"Merriweather";src:local("Merriweather");unicode-range:U+0-FF}@font-face{font-style:bolder;font-family:"Merriweather";src:local("Merriweather");unicode-range:U+0-FF}:not(pre):not(code):not(textarea):not(tt):not(kbd):not(samp):not(var){font-family:"Merriweather"!important}pre,code,textarea,tt,kbd,samp,var{font-family:monospace!important}pre *,code *,textarea *,tt *,kbd *,samp *,var *{font-family:monospace!important}</style>


# <center><span style="color:#000000">Molecular Biosciences Practical Techniques</span></center>

![[Practical 1 PCR 2024.docx]]

## vector
capable of carrying a fragment of foreign DNA  
(Vector + foreign DNA = Recombinant DNA)
cDNA: Complementary DNA
	DNA Without Introns
OPR (Open Reading Frame) And CDS (Coding DNA Sequence) Are The Same Thing

Types of Vectors:
- Cloning Vectors 
	- isolate and amplify DNA
	- ![Pasted image 20251109222050.png](/img/user/images/Pasted%20image%2020251109222050.png)
- Expression Vectors
	- isolate and amplify DNA
	- express foreign DNA (gene) in host organism/cell (DNA RNA Protein)
	- ![Pasted image 20251109220241.png](/img/user/images/Pasted%20image%2020251109220241.png)



Other Vectors:
- Bacteriophage
- Bacterial Artificail Chromosome (BAC)
- Yeast Artificial Chromosome (YAC)

### Plasmid 
 dsDNA molecule capable of independent replication within a host organism (Bacteria).  

All plasmids contain:
- Origin of replication (ori)
- Charicteristic gene (Like amp<sup>r</sup>)

#### Plasmid Modifications
##### Multiple cloning site
A small sequence (~30-80bp) of DNA containing recognition sites for a number of common restriction enzymes (must be unique to the cloning site)

pBluescript II Mcs:
Kpn1-XhoI-ClaI-HindIII-EcoRV-EcoRI-PstI-BamHI-XbaI--NotI-BstxI-SacI

##### screening system System
###### The LacZ selection system
The b-galactosidase gene, LacZ, metabolizes Xgal    Producing a blue product
Insertion of foreign DNA into the MCS destroys the LacZ gene    no blue product

## Gene Cloning

## Restriction Enzymes
Found naturally in bacteria- protect against infection from bacteriophages
Cut DNA at specific, palindromic sites

Size of insert (bp) X amount of vector (ng)
Over
Size of vector (bp)
1:1

 1 unit of restriction enzyme is defined as the amount required to cut 1ug of DNA in 1hr.
Enzymes are not stable and degrade with time
They should be kept on ice at all times and stored @ -20OC.
By reducing the units of enzyme used or time incubated DNA can be partially digested.

## DNA ligase
catalyses the formation of a phosphodiester bond between the 3’-hydroxyl group at the end of one double stranded DNA chain and the 5’-phosphoryl group at the end of another.
To ensure the best chance of a successful ligation reaction we add more insert (SPINT2) than Vector (pCDNA3.1)
When preforming ligations we measure the amount of DNA used in ends:  
100ng of SPINT2 DNA contains 7X more ends than 100ng of pCDNA3.1 DNA

To calculate the ug of an insert required to equal the ends of a vector:

## Primer
Forward Primer Is The Same As The Target Primer
Reverse Primer Is Complimentary To The Forward DNA

Primers Made With 21 Base Pairs

Primer written: 5'  3'  

## SPINT2
SPINT2 Is Also Called HFG Activator Inhibitor Type 2 (HAI-2)
Hepatocyte growth factor (HFG)
MET   (AKA: HGF receptor)

## Transformation
Genetic alteration of a cell following uptake and expression of exogenous genetic material (plasmids).
Naturally occurring: Horizontal gene transfer  
Modified by Biologists used as a powerful tool in research, Biotechnology and medicine

e.Coli are not naturally competent

Methods of transformation:
- Electroporation
- Heat Shock

Host Cell must be sensitive to the antibiotic
The vector carrying Antibiotic resistant gene confers resistant to the host cell

### Scoring successful Transformants (pcDNA3.1 control)

#### Number of transformants per ug of DNA
No. of colonies: 200
Total Volume: 1000ul  
100ul plated  
DNA conc=10ug/ml (/10=ng/ul)  
5ul used=50ng=0.05ug  
0.05ug plasmid added  
Used 100ul of a total 1000ul  
100/1000= 0.1  
0.1X0.05= 0.005ug DNA(=5ng)  
So: 0.005ug gives 200 colonies  
Then: 200 (colonies) X1.0 (ug)/0.005=  
40,000 successful transformants per ug  
plasmid  
(4.0X104 )

### Scoring successful Transformants
#### Number of transforming molecules per ug of plasmid
No. of colonies: 200
Plasmid pCDNA3.1  (5.427kb)
1.5kb= 1X106 Da  
1 pCDNA3.1 molecule has a mass of (5.427/1.5)=  
3.62X106 Da  
1mole of pCDNA3.1= 3.62X106 g  
3.62X106 g= 6.02X1023 molecules  
1ug= 1X10-6 g  
So: 3.62X106 g= 3.62X106 X 1X106 = 3.62X1012 ug  
3.62X1012 ug= 6.02X1023 molecules  
So: 6.02X1023 /3.62X1012 = 1.66X1011 molecules  
of pCDNA3.1/ug  
(0.005ug in 100ul on plate  
0.005 X 1.66X1011 = 8.3X108 molecules  
830,000,000)


#### Number of E.coli plated
No. of colonies: 200
Plasmid pCDNA3.1  
(5.427kb)
Compitent cells conc.=5X108 /ml  
Added 100ul (5X107 ), total vol=1000ul  
Used 100ul:5X107 X (100/1000) 0.1=  
5,000,000= 5X106 cells plated
Fraction of successfully transformed bacteria:  
5X106 /200= 1 in 25,000
8.3X108 plasmids plated (1.66X1011 /ug)  
Number of successful transformants:  
200  
8.3X108 /200= 4.15X106 molecules per  
successful transformation
1 in 4.15X106 Successfully transformed
(4.15X106 = 4.15 million)

### Scoring successful Transformants (Ligation)
#### Number of transformants per ug of DNA
No. of colonies: 200
Plasmid pCDNA3.1
(5.427kb)
+SPINT2 (0.8kb)
=6.23kb
Number of transformants per ug of DNA
Total Volume: 1000ul  
1000ul plated  
DNA conc=100ng of pCDNA3.1 in 20ul  
+15ng of SPINT2=115ng  
10ul used=58ng=0.058ug  
0.058ug plasmid added  
Used 1000ul of a total 1000ul  
So: 0.058ug gives 200 colonies  
Then: 200 (colonies) X1.0 (ug)/0.058=  
3,448 successful transformants per ug  
plasmid  
(3.450X103 )
#### Number of transforming molecules per ug of plasmid
No. of colonies: 200
Plasmid pCDNA3.1
(5.427kb)
+SPINT2 (0.8kb)
=6.23kb
1.5kb= 1X106 Da  
1 pCDNA3.1-SPINT2 molecule has a mass of  
(6.23/1.5)= 4.15X106 Da  
1mole of pCDNA3.1-SPINT2= 4.15X106 g  
4.15X106 g= 6.02X1023 molecules  
1ug= 1X10-6 g  
So: 4.15X106 g= 4.15X106 X 1X106 = 4.15X1012 ug  
4.15X1012 ug= 6.02X1023 molecules  
So: 6.02X1023 /4.15X1012 = 1.45X1011 molecules  
of pCDNA3.1-SPINT2/ug  
(0.058ug in 1000ul on plate  
0.058 X 1.45X1011 = 8.41X109 molecules)
#### Number of E.coli plated
No. of colonies: 200
Plasmid pCDNA3.1
(5.427kb)
+SPINT2 (0.8kb)
=6.23kb
Compitent cells conc.=5X108 /ml
Added 100ul (5X107 ), total vol=1000ul
Used 1000ul = 5X107 cells plated
Fraction of successfully transformed bacteria:
5X107 /200= 1 in 250,000
8.41X109 plasmids plated (1.45X1011 /ug)
Number of successful transformants:
200
8.41X109 /200= 4.2X107 molecules per
successful transformation
1 in 4.2X107 Successfully transformed
(4.2X107 = 42 million)

## Moles, Daltons and Constants
A Mole is the amount of a substance that contains the same number of elementary entities (atoms, molecules etc.) as 12g of Carbon.

Atomic Mass of Carbon= 12Da (The accepted unit of molecular Mass is the Dalton (Da or kDa)
1Mol of Carbon= 12g

The number of Carbon atoms in 1Mol (12g) is:  6.022 X1023 
This is the Avogadro constant: N<sub>A</sub> or L  
N<sub>A</sub> is the number if units of a substance per Mole.  
6.022 X1023 mol-1
### Calculations
Atomic Mass of Carbon= 12  
1Mol of Carbon= 12g  
6.022 X1023 atoms in 12g= 1mol  

Atomic Mass of Sodium= 22.99  
1Mol of Sodium= 22.99g  
6.022 X1023 atoms in 22.99g= 1mol

Atomic Mass of Sodium= 22.99  
Atomic Mass of Oxygen= 16  
Atomic Mass of hydrogen= 1.01  
Atomic Mass of sodium Hydroxide= 40.00  
1Mol of Sodium Hydroxide= 40.00g  
6.022 X1023 Molecules in 40.00g

The Molecular Mass of dsDNA  
1 Base pair approx. Average Mass:  
650Da  
1500bp (1.5kb) X650=975,000 Da  
=0.975X106 Da  
1X106 Da  
1mol= 1X106g =1000kg  
6.022X1023 Molecules in 1000kg  
6.022X1020 Molecules in 1kg (1000g)  
6.022X1017 Molecules in 1g  
6.022X1014 Molecules in 1mg (0.001g)  
6.022X1011 Molecules in 1ug (0.000001g)  
602,200,000,000  
(6.022X1023/1X1012=6.022X1011)

#### How many molecules of pBS are there in 100ng?
 1.5kb= 1X106 Da
(3/1.5=2)
 1 pBS molecule has a mass of (3/1.5)= 2X106 Da
1mole of pBS= 2X106g
2X106g= 6.02X1023molecules
1ug= 1X10-6g
So: 2X106g= 2X106 X 1X106= 2X1012ug
2X1012ug= 6.02X1023molecules
So: 6.02X1023/2X1012= 3X1011 molecules of pBS/ug
1000ng in 1ug: 3X1011/1000=3X108 molecules of pBS in 1ng

1ng= 1X10-9g  
So: 2X106g= 2X106 X 1X109= 2X1015ng

6.02X1023/2X1015=3X108 (1ng) X100= 3X1010 (100ng)

## Expression of cloned genes in human cells
Cellular localisation [images]
Effect on Cell physiology
## Misc
![Pasted image 20251109213326.png](/img/user/images/Pasted%20image%2020251109213326.png)
![Pasted image 20251109213332.png](/img/user/images/Pasted%20image%2020251109213332.png)
![Pasted image 20251109213338.png](/img/user/images/Pasted%20image%2020251109213338.png)
![Pasted image 20251109213401.png](/img/user/images/Pasted%20image%2020251109213401.png)
![Pasted image 20251109213525.png](/img/user/images/Pasted%20image%2020251109213525.png)
![Pasted image 20251109213621.png](/img/user/images/Pasted%20image%2020251109213621.png)
![Pasted image 20251109215235.png](/img/user/images/Pasted%20image%2020251109215235.png)
![Pasted image 20251109220206.png](/img/user/images/Pasted%20image%2020251109220206.png)
![Pasted image 20251109220215.png](/img/user/images/Pasted%20image%2020251109220215.png)
![Pasted image 20251109222044.png](/img/user/images/Pasted%20image%2020251109222044.png)
![Pasted image 20251109215943.png](/img/user/images/Pasted%20image%2020251109215943.png)
![Pasted image 20251109220003.png](/img/user/images/Pasted%20image%2020251109220003.png)
![Pasted image 20251109220012.png](/img/user/images/Pasted%20image%2020251109220012.png)
![Pasted image 20251109220026.png](/img/user/images/Pasted%20image%2020251109220026.png)
![Pasted image 20251109220326.png](/img/user/images/Pasted%20image%2020251109220326.png)
![Pasted image 20251109220333.png](/img/user/images/Pasted%20image%2020251109220333.png)
![Pasted image 20251109220612.png](/img/user/images/Pasted%20image%2020251109220612.png)
![Pasted image 20251109220729.png](/img/user/images/Pasted%20image%2020251109220729.png)
![Pasted image 20251109221116.png](/img/user/images/Pasted%20image%2020251109221116.png)
![Pasted image 20251109221356.png](/img/user/images/Pasted%20image%2020251109221356.png)
![Pasted image 20251109221428.png](/img/user/images/Pasted%20image%2020251109221428.png)
![Pasted image 20251109221820.png](/img/user/images/Pasted%20image%2020251109221820.png)
![Pasted image 20251109222235.png](/img/user/images/Pasted%20image%2020251109222235.png)
![Pasted image 20251109222300.png](/img/user/images/Pasted%20image%2020251109222300.png)
![Pasted image 20251109223041.png](/img/user/images/Pasted%20image%2020251109223041.png)
![Pasted image 20251109223046.png](/img/user/images/Pasted%20image%2020251109223046.png)
![Pasted image 20251109223117.png](/img/user/images/Pasted%20image%2020251109223117.png)
![Pasted image 20251109223122.png](/img/user/images/Pasted%20image%2020251109223122.png)
![Pasted image 20251109223134.png](/img/user/images/Pasted%20image%2020251109223134.png)

## Stuff I Missed
![Pasted image 20251109213413.png](/img/user/images/Pasted%20image%2020251109213413.png)
![Pasted image 20251109213423.png](/img/user/images/Pasted%20image%2020251109213423.png)
![Pasted image 20251109214106.png](/img/user/images/Pasted%20image%2020251109214106.png)
![Pasted image 20251109214222.png](/img/user/images/Pasted%20image%2020251109214222.png)

![Pasted image 20251109214703.png](/img/user/images/Pasted%20image%2020251109214703.png)
![Pasted image 20251109214737.png](/img/user/images/Pasted%20image%2020251109214737.png)
![Pasted image 20251109214757.png](/img/user/images/Pasted%20image%2020251109214757.png)
![Pasted image 20251109215257.png](/img/user/images/Pasted%20image%2020251109215257.png)
![Pasted image 20251109215303.png](/img/user/images/Pasted%20image%2020251109215303.png)
![Pasted image 20251109215309.png](/img/user/images/Pasted%20image%2020251109215309.png)
![Pasted image 20251109222528.png](/img/user/images/Pasted%20image%2020251109222528.png)
![Pasted image 20251109222532.png](/img/user/images/Pasted%20image%2020251109222532.png)



Definitions 

Complimentary DNA (cDNA): DNA synthesised from a mRNA template catalysed by the retroviral enzyme reverse transcriptase.

Oligonucleotide: a short nucleic acid polymer, in vivo these can be generated by enzymes such as primase. Commercially, they are generated in vitro by polymerising individual nucleotide precursors.


Open Reading Frame (ORF): the portion of a gene that comprises the coding base pair triplets for a protein and nothing else (all introns, 5’ and 3’ non-coding regions have been removed). 

Primase: Enzymes whose continual activity is required at the DNA replication fork. They catalyze the synthesis of short RNA molecules used as primers for DNA polymerases to extend new daughter DNA strands from.

Reverse Transcriptase: An enzyme used to generate DNA from an RNA template. The enzyme is encoded by reverse-transcribing RNA viruses (e.g. retroviruses) to convert their RNA genomes into DNA  

	








<center><sub>Done :)</sub></center>